2023-10-28 13:20:50,390 - INFO - {'e': 'aggTrade', 'E': 1698479449907, 's': 'BTCUSDT', 'a': 2740901125, 'p': '34066.67000000', 'q': '0.00066000', 'f': 3260167588, 'l': 3260167588, 'T': 1698479449906, 'm': True, 'M': True}
2023-10-28 13:20:50,399 - INFO - {'e': 'aggTrade', 'E': 1698479449907, 's': 'BTCUSDT', 'a': 2740901125, 'p': '34066.67000000', 'q': '0.00066000', 'f': 3260167588, 'l': 3260167588, 'T': 1698479449906, 'm': True, 'M': True}
2023-10-28 13:20:52,438 - INFO - {'e': 'aggTrade', 'E': 1698479453058, 's': 'BTCUSDT', 'a': 2740901130, 'p': '34066.67000000', 'q': '0.00707000', 'f': 3260167598, 'l': 3260167598, 'T': 1698479453057, 'm': True, 'M': True}
2023-10-28 13:20:52,452 - INFO - {'e': 'aggTrade', 'E': 1698479453058, 's': 'BTCUSDT', 'a': 2740901130, 'p': '34066.67000000', 'q': '0.00707000', 'f': 3260167598, 'l': 3260167598, 'T': 1698479453057, 'm': True, 'M': True}
2023-10-28 13:20:54,514 - INFO - {'e': 'aggTrade', 'E': 1698479455002, 's': 'BTCUSDT', 'a': 2740901140, 'p': '34066.67000000', 'q': '0.00089000', 'f': 3260167617, 'l': 3260167617, 'T': 1698479455001, 'm': True, 'M': True}
2023-10-28 13:20:54,514 - INFO - {'e': 'aggTrade', 'E': 1698479455002, 's': 'BTCUSDT', 'a': 2740901140, 'p': '34066.67000000', 'q': '0.00089000', 'f': 3260167617, 'l': 3260167617, 'T': 1698479455001, 'm': True, 'M': True}
2023-10-28 13:20:56,591 - INFO - {'e': 'aggTrade', 'E': 1698479457362, 's': 'BTCUSDT', 'a': 2740901150, 'p': '34066.67000000', 'q': '0.00019000', 'f': 3260167633, 'l': 3260167633, 'T': 1698479457361, 'm': True, 'M': True}
2023-10-28 13:20:56,602 - INFO - {'e': 'aggTrade', 'E': 1698479457362, 's': 'BTCUSDT', 'a': 2740901150, 'p': '34066.67000000', 'q': '0.00019000', 'f': 3260167633, 'l': 3260167633, 'T': 1698479457361, 'm': True, 'M': True}
2023-10-28 13:20:58,669 - INFO - {'e': 'aggTrade', 'E': 1698479458375, 's': 'BTCUSDT', 'a': 2740901154, 'p': '34066.67000000', 'q': '0.00059000', 'f': 3260167637, 'l': 3260167637, 'T': 1698479458374, 'm': True, 'M': True}
2023-10-28 13:20:58,669 - INFO - {'e': 'aggTrade', 'E': 1698479458375, 's': 'BTCUSDT', 'a': 2740901154, 'p': '34066.67000000', 'q': '0.00059000', 'f': 3260167637, 'l': 3260167637, 'T': 1698479458374, 'm': True, 'M': True}
2023-10-28 13:21:00,731 - INFO - {'e': 'aggTrade', 'E': 1698479461375, 's': 'BTCUSDT', 'a': 2740901217, 'p': '34066.55000000', 'q': '0.00089000', 'f': 3260167720, 'l': 3260167720, 'T': 1698479461375, 'm': True, 'M': True}
2023-10-28 13:21:00,731 - INFO - {'e': 'aggTrade', 'E': 1698479461375, 's': 'BTCUSDT', 'a': 2740901217, 'p': '34066.55000000', 'q': '0.00089000', 'f': 3260167720, 'l': 3260167720, 'T': 1698479461375, 'm': True, 'M': True}
2023-10-28 13:21:02,771 - INFO - {'e': 'aggTrade', 'E': 1698479461765, 's': 'BTCUSDT', 'a': 2740901218, 'p': '34066.55000000', 'q': '0.00029000', 'f': 3260167721, 'l': 3260167721, 'T': 1698479461765, 'm': True, 'M': True}
2023-10-28 13:21:02,771 - INFO - {'e': 'aggTrade', 'E': 1698479461765, 's': 'BTCUSDT', 'a': 2740901218, 'p': '34066.55000000', 'q': '0.00029000', 'f': 3260167721, 'l': 3260167721, 'T': 1698479461765, 'm': True, 'M': True}
2023-10-28 13:21:04,870 - INFO - {'e': 'aggTrade', 'E': 1698479464637, 's': 'BTCUSDT', 'a': 2740901219, 'p': '34066.56000000', 'q': '0.00248000', 'f': 3260167722, 'l': 3260167722, 'T': 1698479464637, 'm': False, 'M': True}
2023-10-28 13:21:04,870 - INFO - {'e': 'aggTrade', 'E': 1698479464637, 's': 'BTCUSDT', 'a': 2740901219, 'p': '34066.56000000', 'q': '0.00248000', 'f': 3260167722, 'l': 3260167722, 'T': 1698479464637, 'm': False, 'M': True}
2023-10-28 13:21:06,947 - INFO - {'e': 'aggTrade', 'E': 1698479467623, 's': 'BTCUSDT', 'a': 2740901225, 'p': '34066.55000000', 'q': '0.00294000', 'f': 3260167729, 'l': 3260167729, 'T': 1698479467622, 'm': True, 'M': True}
2023-10-28 13:21:06,947 - INFO - {'e': 'aggTrade', 'E': 1698479467623, 's': 'BTCUSDT', 'a': 2740901225, 'p': '34066.55000000', 'q': '0.00294000', 'f': 3260167729, 'l': 3260167729, 'T': 1698479467622, 'm': True, 'M': True}
2023-10-28 13:21:09,016 - INFO - {'e': 'aggTrade', 'E': 1698479468578, 's': 'BTCUSDT', 'a': 2740901226, 'p': '34066.55000000', 'q': '0.00289000', 'f': 3260167730, 'l': 3260167730, 'T': 1698479468577, 'm': True, 'M': True}
2023-10-28 13:21:09,016 - INFO - {'e': 'aggTrade', 'E': 1698479468578, 's': 'BTCUSDT', 'a': 2740901226, 'p': '34066.55000000', 'q': '0.00289000', 'f': 3260167730, 'l': 3260167730, 'T': 1698479468577, 'm': True, 'M': True}
2023-10-28 13:21:11,080 - INFO - {'e': 'aggTrade', 'E': 1698479468578, 's': 'BTCUSDT', 'a': 2740901226, 'p': '34066.55000000', 'q': '0.00289000', 'f': 3260167730, 'l': 3260167730, 'T': 1698479468577, 'm': True, 'M': True}
2023-10-28 13:21:11,080 - INFO - {'e': 'aggTrade', 'E': 1698479468578, 's': 'BTCUSDT', 'a': 2740901226, 'p': '34066.55000000', 'q': '0.00289000', 'f': 3260167730, 'l': 3260167730, 'T': 1698479468577, 'm': True, 'M': True}
2023-10-28 13:21:13,132 - INFO - {'e': 'aggTrade', 'E': 1698479473916, 's': 'BTCUSDT', 'a': 2740901232, 'p': '34066.56000000', 'q': '0.00425000', 'f': 3260167736, 'l': 3260167736, 'T': 1698479473915, 'm': False, 'M': True}
2023-10-28 13:21:13,132 - INFO - {'e': 'aggTrade', 'E': 1698479473916, 's': 'BTCUSDT', 'a': 2740901232, 'p': '34066.56000000', 'q': '0.00425000', 'f': 3260167736, 'l': 3260167736, 'T': 1698479473915, 'm': False, 'M': True}
2023-10-28 13:21:15,185 - INFO - {'e': 'aggTrade', 'E': 1698479475815, 's': 'BTCUSDT', 'a': 2740901234, 'p': '34066.56000000', 'q': '0.05217000', 'f': 3260167738, 'l': 3260167738, 'T': 1698479475815, 'm': False, 'M': True}
2023-10-28 13:21:15,187 - INFO - {'e': 'aggTrade', 'E': 1698479475815, 's': 'BTCUSDT', 'a': 2740901234, 'p': '34066.56000000', 'q': '0.05217000', 'f': 3260167738, 'l': 3260167738, 'T': 1698479475815, 'm': False, 'M': True}
2023-10-28 13:21:17,226 - INFO - {'e': 'aggTrade', 'E': 1698479477557, 's': 'BTCUSDT', 'a': 2740901236, 'p': '34066.56000000', 'q': '0.02713000', 'f': 3260167740, 'l': 3260167740, 'T': 1698479477557, 'm': False, 'M': True}
2023-10-28 13:21:17,226 - INFO - {'e': 'aggTrade', 'E': 1698479477557, 's': 'BTCUSDT', 'a': 2740901236, 'p': '34066.56000000', 'q': '0.02713000', 'f': 3260167740, 'l': 3260167740, 'T': 1698479477557, 'm': False, 'M': True}
2023-10-28 13:21:19,274 - INFO - {'e': 'aggTrade', 'E': 1698479478744, 's': 'BTCUSDT', 'a': 2740901243, 'p': '34066.39000000', 'q': '0.03837000', 'f': 3260167751, 'l': 3260167751, 'T': 1698479478744, 'm': False, 'M': True}
2023-10-28 13:21:19,274 - INFO - {'e': 'aggTrade', 'E': 1698479478744, 's': 'BTCUSDT', 'a': 2740901243, 'p': '34066.39000000', 'q': '0.03837000', 'f': 3260167751, 'l': 3260167751, 'T': 1698479478744, 'm': False, 'M': True}
2023-10-28 13:21:21,350 - INFO - {'e': 'aggTrade', 'E': 1698479481960, 's': 'BTCUSDT', 'a': 2740901245, 'p': '34066.39000000', 'q': '0.00100000', 'f': 3260167753, 'l': 3260167753, 'T': 1698479481959, 'm': False, 'M': True}
2023-10-28 13:21:21,352 - INFO - {'e': 'aggTrade', 'E': 1698479481960, 's': 'BTCUSDT', 'a': 2740901245, 'p': '34066.39000000', 'q': '0.00100000', 'f': 3260167753, 'l': 3260167753, 'T': 1698479481959, 'm': False, 'M': True}
2023-10-28 13:21:23,403 - INFO - {'e': 'aggTrade', 'E': 1698479483128, 's': 'BTCUSDT', 'a': 2740901249, 'p': '34066.39000000', 'q': '0.00100000', 'f': 3260167757, 'l': 3260167757, 'T': 1698479483128, 'm': False, 'M': True}
2023-10-28 13:21:23,403 - INFO - {'e': 'aggTrade', 'E': 1698479483128, 's': 'BTCUSDT', 'a': 2740901249, 'p': '34066.39000000', 'q': '0.00100000', 'f': 3260167757, 'l': 3260167757, 'T': 1698479483128, 'm': False, 'M': True}
2023-10-28 13:21:25,433 - INFO - {'e': 'aggTrade', 'E': 1698479485879, 's': 'BTCUSDT', 'a': 2740901251, 'p': '34066.39000000', 'q': '0.00048000', 'f': 3260167759, 'l': 3260167759, 'T': 1698479485879, 'm': False, 'M': True}
2023-10-28 13:21:25,433 - INFO - {'e': 'aggTrade', 'E': 1698479485879, 's': 'BTCUSDT', 'a': 2740901251, 'p': '34066.39000000', 'q': '0.00048000', 'f': 3260167759, 'l': 3260167759, 'T': 1698479485879, 'm': False, 'M': True}
2023-10-28 13:21:27,503 - INFO - {'e': 'aggTrade', 'E': 1698479487784, 's': 'BTCUSDT', 'a': 2740901253, 'p': '34066.39000000', 'q': '0.01204000', 'f': 3260167761, 'l': 3260167761, 'T': 1698479487783, 'm': False, 'M': True}
2023-10-28 13:21:27,503 - INFO - {'e': 'aggTrade', 'E': 1698479487784, 's': 'BTCUSDT', 'a': 2740901253, 'p': '34066.39000000', 'q': '0.01204000', 'f': 3260167761, 'l': 3260167761, 'T': 1698479487783, 'm': False, 'M': True}
2023-10-28 13:21:29,597 - INFO - {'e': 'aggTrade', 'E': 1698479490000, 's': 'BTCUSDT', 'a': 2740901256, 'p': '34066.38000000', 'q': '0.00733000', 'f': 3260167767, 'l': 3260167767, 'T': 1698479489999, 'm': True, 'M': True}
2023-10-28 13:21:29,597 - INFO - {'e': 'aggTrade', 'E': 1698479490000, 's': 'BTCUSDT', 'a': 2740901256, 'p': '34066.38000000', 'q': '0.00733000', 'f': 3260167767, 'l': 3260167767, 'T': 1698479489999, 'm': True, 'M': True}
2023-10-28 13:21:31,662 - INFO - {'e': 'aggTrade', 'E': 1698479492465, 's': 'BTCUSDT', 'a': 2740901259, 'p': '34066.38000000', 'q': '0.00250000', 'f': 3260167770, 'l': 3260167770, 'T': 1698479492465, 'm': True, 'M': True}
2023-10-28 13:21:31,662 - INFO - {'e': 'aggTrade', 'E': 1698479492465, 's': 'BTCUSDT', 'a': 2740901259, 'p': '34066.38000000', 'q': '0.00250000', 'f': 3260167770, 'l': 3260167770, 'T': 1698479492465, 'm': True, 'M': True}
2023-10-28 13:21:33,719 - INFO - {'e': 'aggTrade', 'E': 1698479494085, 's': 'BTCUSDT', 'a': 2740901262, 'p': '34066.38000000', 'q': '0.02935000', 'f': 3260167773, 'l': 3260167773, 'T': 1698479494084, 'm': True, 'M': True}
2023-10-28 13:21:33,725 - INFO - {'e': 'aggTrade', 'E': 1698479494085, 's': 'BTCUSDT', 'a': 2740901262, 'p': '34066.38000000', 'q': '0.02935000', 'f': 3260167773, 'l': 3260167773, 'T': 1698479494084, 'm': True, 'M': True}
2023-10-28 13:21:35,778 - INFO - {'e': 'aggTrade', 'E': 1698479495877, 's': 'BTCUSDT', 'a': 2740901268, 'p': '34066.39000000', 'q': '0.00060000', 'f': 3260167779, 'l': 3260167779, 'T': 1698479495877, 'm': False, 'M': True}
2023-10-28 13:21:35,778 - INFO - {'e': 'aggTrade', 'E': 1698479495877, 's': 'BTCUSDT', 'a': 2740901268, 'p': '34066.39000000', 'q': '0.00060000', 'f': 3260167779, 'l': 3260167779, 'T': 1698479495877, 'm': False, 'M': True}
2023-10-28 13:21:37,842 - INFO - {'e': 'aggTrade', 'E': 1698479498548, 's': 'BTCUSDT', 'a': 2740901269, 'p': '34066.38000000', 'q': '0.00021000', 'f': 3260167780, 'l': 3260167780, 'T': 1698479498547, 'm': True, 'M': True}
2023-10-28 13:21:37,842 - INFO - {'e': 'aggTrade', 'E': 1698479498548, 's': 'BTCUSDT', 'a': 2740901269, 'p': '34066.38000000', 'q': '0.00021000', 'f': 3260167780, 'l': 3260167780, 'T': 1698479498547, 'm': True, 'M': True}
2023-10-28 13:21:41,465 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 13:21:41,467 - ERROR - No data available
2023-10-28 13:21:43,519 - INFO - {'e': 'aggTrade', 'E': 1698479503937, 's': 'BTCUSDT', 'a': 2740901278, 'p': '34066.39000000', 'q': '0.00292000', 'f': 3260167789, 'l': 3260167789, 'T': 1698479503937, 'm': False, 'M': True}
2023-10-28 13:21:43,519 - INFO - {'e': 'aggTrade', 'E': 1698479503937, 's': 'BTCUSDT', 'a': 2740901278, 'p': '34066.39000000', 'q': '0.00292000', 'f': 3260167789, 'l': 3260167789, 'T': 1698479503937, 'm': False, 'M': True}
2023-10-28 13:21:45,585 - INFO - {'e': 'aggTrade', 'E': 1698479505722, 's': 'BTCUSDT', 'a': 2740901283, 'p': '34066.38000000', 'q': '0.00082000', 'f': 3260167794, 'l': 3260167794, 'T': 1698479505722, 'm': True, 'M': True}
2023-10-28 13:21:45,590 - INFO - {'e': 'aggTrade', 'E': 1698479505722, 's': 'BTCUSDT', 'a': 2740901283, 'p': '34066.38000000', 'q': '0.00082000', 'f': 3260167794, 'l': 3260167794, 'T': 1698479505722, 'm': True, 'M': True}
2023-10-28 13:21:47,667 - INFO - {'e': 'aggTrade', 'E': 1698479508052, 's': 'BTCUSDT', 'a': 2740901286, 'p': '34066.38000000', 'q': '0.01468000', 'f': 3260167797, 'l': 3260167797, 'T': 1698479508051, 'm': True, 'M': True}
2023-10-28 13:21:47,667 - INFO - {'e': 'aggTrade', 'E': 1698479508052, 's': 'BTCUSDT', 'a': 2740901286, 'p': '34066.38000000', 'q': '0.01468000', 'f': 3260167797, 'l': 3260167797, 'T': 1698479508051, 'm': True, 'M': True}
2023-10-28 13:21:49,716 - INFO - {'e': 'aggTrade', 'E': 1698479510380, 's': 'BTCUSDT', 'a': 2740901299, 'p': '34066.37000000', 'q': '0.00676000', 'f': 3260167833, 'l': 3260167833, 'T': 1698479510379, 'm': True, 'M': True}
2023-10-28 13:21:49,721 - INFO - {'e': 'aggTrade', 'E': 1698479510380, 's': 'BTCUSDT', 'a': 2740901299, 'p': '34066.37000000', 'q': '0.00676000', 'f': 3260167833, 'l': 3260167833, 'T': 1698479510379, 'm': True, 'M': True}
2023-10-28 13:21:51,775 - INFO - {'e': 'aggTrade', 'E': 1698479511351, 's': 'BTCUSDT', 'a': 2740901300, 'p': '34066.38000000', 'q': '0.00200000', 'f': 3260167834, 'l': 3260167834, 'T': 1698479511350, 'm': False, 'M': True}
2023-10-28 13:21:51,775 - INFO - {'e': 'aggTrade', 'E': 1698479511351, 's': 'BTCUSDT', 'a': 2740901300, 'p': '34066.38000000', 'q': '0.00200000', 'f': 3260167834, 'l': 3260167834, 'T': 1698479511350, 'm': False, 'M': True}
2023-10-28 13:21:53,840 - INFO - {'e': 'aggTrade', 'E': 1698479514607, 's': 'BTCUSDT', 'a': 2740901303, 'p': '34066.38000000', 'q': '0.00077000', 'f': 3260167837, 'l': 3260167837, 'T': 1698479514607, 'm': False, 'M': True}
2023-10-28 13:21:53,840 - INFO - {'e': 'aggTrade', 'E': 1698479514607, 's': 'BTCUSDT', 'a': 2740901303, 'p': '34066.38000000', 'q': '0.00077000', 'f': 3260167837, 'l': 3260167837, 'T': 1698479514607, 'm': False, 'M': True}
2023-10-28 13:21:55,927 - INFO - {'e': 'aggTrade', 'E': 1698479515457, 's': 'BTCUSDT', 'a': 2740901304, 'p': '34066.38000000', 'q': '0.00017000', 'f': 3260167838, 'l': 3260167838, 'T': 1698479515457, 'm': False, 'M': True}
2023-10-28 13:21:55,927 - INFO - {'e': 'aggTrade', 'E': 1698479515457, 's': 'BTCUSDT', 'a': 2740901304, 'p': '34066.38000000', 'q': '0.00017000', 'f': 3260167838, 'l': 3260167838, 'T': 1698479515457, 'm': False, 'M': True}
2023-10-28 13:21:57,983 - INFO - {'e': 'aggTrade', 'E': 1698479518078, 's': 'BTCUSDT', 'a': 2740901309, 'p': '34066.38000000', 'q': '0.00077000', 'f': 3260167843, 'l': 3260167843, 'T': 1698479518077, 'm': False, 'M': True}
2023-10-28 13:21:57,983 - INFO - {'e': 'aggTrade', 'E': 1698479518078, 's': 'BTCUSDT', 'a': 2740901309, 'p': '34066.38000000', 'q': '0.00077000', 'f': 3260167843, 'l': 3260167843, 'T': 1698479518077, 'm': False, 'M': True}
2023-10-28 13:22:00,082 - INFO - {'e': 'aggTrade', 'E': 1698479520289, 's': 'BTCUSDT', 'a': 2740901351, 'p': '34066.37000000', 'q': '0.00033000', 'f': 3260167890, 'l': 3260167890, 'T': 1698479520288, 'm': True, 'M': True}
2023-10-28 13:22:00,082 - INFO - {'e': 'aggTrade', 'E': 1698479520289, 's': 'BTCUSDT', 'a': 2740901351, 'p': '34066.37000000', 'q': '0.00033000', 'f': 3260167890, 'l': 3260167890, 'T': 1698479520288, 'm': True, 'M': True}
2023-10-28 13:22:02,160 - INFO - {'e': 'aggTrade', 'E': 1698479522252, 's': 'BTCUSDT', 'a': 2740901354, 'p': '34066.38000000', 'q': '0.00029000', 'f': 3260167893, 'l': 3260167893, 'T': 1698479522251, 'm': False, 'M': True}
2023-10-28 13:22:02,163 - INFO - {'e': 'aggTrade', 'E': 1698479522252, 's': 'BTCUSDT', 'a': 2740901354, 'p': '34066.38000000', 'q': '0.00029000', 'f': 3260167893, 'l': 3260167893, 'T': 1698479522251, 'm': False, 'M': True}
2023-10-28 13:22:04,212 - INFO - {'e': 'aggTrade', 'E': 1698479524852, 's': 'BTCUSDT', 'a': 2740901355, 'p': '34066.37000000', 'q': '0.00370000', 'f': 3260167894, 'l': 3260167894, 'T': 1698479524851, 'm': True, 'M': True}
2023-10-28 13:22:04,212 - INFO - {'e': 'aggTrade', 'E': 1698479524852, 's': 'BTCUSDT', 'a': 2740901355, 'p': '34066.37000000', 'q': '0.00370000', 'f': 3260167894, 'l': 3260167894, 'T': 1698479524851, 'm': True, 'M': True}
2023-10-28 13:22:06,284 - INFO - {'e': 'aggTrade', 'E': 1698479526726, 's': 'BTCUSDT', 'a': 2740901364, 'p': '34066.37000000', 'q': '0.00069000', 'f': 3260167903, 'l': 3260167903, 'T': 1698479526726, 'm': True, 'M': True}
2023-10-28 13:22:06,284 - INFO - {'e': 'aggTrade', 'E': 1698479526726, 's': 'BTCUSDT', 'a': 2740901364, 'p': '34066.37000000', 'q': '0.00069000', 'f': 3260167903, 'l': 3260167903, 'T': 1698479526726, 'm': True, 'M': True}
2023-10-28 13:22:08,348 - INFO - {'e': 'aggTrade', 'E': 1698479528335, 's': 'BTCUSDT', 'a': 2740901366, 'p': '34066.38000000', 'q': '0.00039000', 'f': 3260167905, 'l': 3260167905, 'T': 1698479528335, 'm': False, 'M': True}
2023-10-28 13:22:08,348 - INFO - {'e': 'aggTrade', 'E': 1698479528335, 's': 'BTCUSDT', 'a': 2740901366, 'p': '34066.38000000', 'q': '0.00039000', 'f': 3260167905, 'l': 3260167905, 'T': 1698479528335, 'm': False, 'M': True}
2023-10-28 13:22:10,426 - INFO - {'e': 'aggTrade', 'E': 1698479531035, 's': 'BTCUSDT', 'a': 2740901372, 'p': '34066.37000000', 'q': '0.00151000', 'f': 3260167911, 'l': 3260167911, 'T': 1698479531035, 'm': True, 'M': True}
2023-10-28 13:22:10,426 - INFO - {'e': 'aggTrade', 'E': 1698479531035, 's': 'BTCUSDT', 'a': 2740901372, 'p': '34066.37000000', 'q': '0.00151000', 'f': 3260167911, 'l': 3260167911, 'T': 1698479531035, 'm': True, 'M': True}
2023-10-28 13:22:12,470 - INFO - {'e': 'aggTrade', 'E': 1698479533225, 's': 'BTCUSDT', 'a': 2740901375, 'p': '34066.38000000', 'q': '0.01327000', 'f': 3260167914, 'l': 3260167914, 'T': 1698479533224, 'm': False, 'M': True}
2023-10-28 13:22:12,471 - INFO - {'e': 'aggTrade', 'E': 1698479533225, 's': 'BTCUSDT', 'a': 2740901375, 'p': '34066.38000000', 'q': '0.01327000', 'f': 3260167914, 'l': 3260167914, 'T': 1698479533224, 'm': False, 'M': True}
2023-10-28 13:22:14,511 - INFO - {'e': 'aggTrade', 'E': 1698479535163, 's': 'BTCUSDT', 'a': 2740901379, 'p': '34066.38000000', 'q': '0.00096000', 'f': 3260167918, 'l': 3260167918, 'T': 1698479535162, 'm': False, 'M': True}
2023-10-28 13:22:14,513 - INFO - {'e': 'aggTrade', 'E': 1698479535163, 's': 'BTCUSDT', 'a': 2740901379, 'p': '34066.38000000', 'q': '0.00096000', 'f': 3260167918, 'l': 3260167918, 'T': 1698479535162, 'm': False, 'M': True}
2023-10-28 13:22:16,583 - INFO - {'e': 'aggTrade', 'E': 1698479537384, 's': 'BTCUSDT', 'a': 2740901386, 'p': '34066.37000000', 'q': '0.00064000', 'f': 3260167927, 'l': 3260167928, 'T': 1698479537383, 'm': True, 'M': True}
2023-10-28 13:22:16,583 - INFO - {'e': 'aggTrade', 'E': 1698479537384, 's': 'BTCUSDT', 'a': 2740901386, 'p': '34066.37000000', 'q': '0.00064000', 'f': 3260167927, 'l': 3260167928, 'T': 1698479537383, 'm': True, 'M': True}
2023-10-28 13:22:18,630 - INFO - {'e': 'aggTrade', 'E': 1698479539241, 's': 'BTCUSDT', 'a': 2740901392, 'p': '34066.37000000', 'q': '0.01467000', 'f': 3260167936, 'l': 3260167937, 'T': 1698479539240, 'm': True, 'M': True}
2023-10-28 13:22:18,630 - INFO - {'e': 'aggTrade', 'E': 1698479539241, 's': 'BTCUSDT', 'a': 2740901392, 'p': '34066.37000000', 'q': '0.01467000', 'f': 3260167936, 'l': 3260167937, 'T': 1698479539240, 'm': True, 'M': True}
2023-10-28 13:22:20,696 - INFO - {'e': 'aggTrade', 'E': 1698479540352, 's': 'BTCUSDT', 'a': 2740901394, 'p': '34066.37000000', 'q': '0.00766000', 'f': 3260167939, 'l': 3260167940, 'T': 1698479540351, 'm': True, 'M': True}
2023-10-28 13:22:20,700 - INFO - {'e': 'aggTrade', 'E': 1698479540352, 's': 'BTCUSDT', 'a': 2740901394, 'p': '34066.37000000', 'q': '0.00766000', 'f': 3260167939, 'l': 3260167940, 'T': 1698479540351, 'm': True, 'M': True}
2023-10-28 13:22:22,743 - INFO - {'e': 'aggTrade', 'E': 1698479543255, 's': 'BTCUSDT', 'a': 2740901403, 'p': '34066.37000000', 'q': '0.03983000', 'f': 3260167959, 'l': 3260167963, 'T': 1698479543254, 'm': True, 'M': True}
2023-10-28 13:22:22,745 - INFO - {'e': 'aggTrade', 'E': 1698479543255, 's': 'BTCUSDT', 'a': 2740901403, 'p': '34066.37000000', 'q': '0.03983000', 'f': 3260167959, 'l': 3260167963, 'T': 1698479543254, 'm': True, 'M': True}
2023-10-28 13:22:24,798 - INFO - {'e': 'aggTrade', 'E': 1698479544929, 's': 'BTCUSDT', 'a': 2740901410, 'p': '34066.20000000', 'q': '0.00212000', 'f': 3260167970, 'l': 3260167970, 'T': 1698479544929, 'm': True, 'M': True}
2023-10-28 13:22:24,798 - INFO - {'e': 'aggTrade', 'E': 1698479544929, 's': 'BTCUSDT', 'a': 2740901410, 'p': '34066.20000000', 'q': '0.00212000', 'f': 3260167970, 'l': 3260167970, 'T': 1698479544929, 'm': True, 'M': True}
2023-10-28 13:22:26,848 - INFO - {'e': 'aggTrade', 'E': 1698479546084, 's': 'BTCUSDT', 'a': 2740901412, 'p': '34066.18000000', 'q': '0.01420000', 'f': 3260167972, 'l': 3260167972, 'T': 1698479546083, 'm': True, 'M': True}
2023-10-28 13:22:26,848 - INFO - {'e': 'aggTrade', 'E': 1698479546084, 's': 'BTCUSDT', 'a': 2740901412, 'p': '34066.18000000', 'q': '0.01420000', 'f': 3260167972, 'l': 3260167972, 'T': 1698479546083, 'm': True, 'M': True}
2023-10-28 13:22:28,895 - INFO - {'e': 'aggTrade', 'E': 1698479549632, 's': 'BTCUSDT', 'a': 2740901418, 'p': '34066.18000000', 'q': '0.00015000', 'f': 3260167978, 'l': 3260167978, 'T': 1698479549631, 'm': True, 'M': True}
2023-10-28 13:22:28,895 - INFO - {'e': 'aggTrade', 'E': 1698479549632, 's': 'BTCUSDT', 'a': 2740901418, 'p': '34066.18000000', 'q': '0.00015000', 'f': 3260167978, 'l': 3260167978, 'T': 1698479549631, 'm': True, 'M': True}
2023-10-28 13:22:30,941 - INFO - {'e': 'aggTrade', 'E': 1698479551490, 's': 'BTCUSDT', 'a': 2740901422, 'p': '34066.18000000', 'q': '0.00082000', 'f': 3260167982, 'l': 3260167982, 'T': 1698479551490, 'm': True, 'M': True}
2023-10-28 13:22:30,944 - INFO - {'e': 'aggTrade', 'E': 1698479551490, 's': 'BTCUSDT', 'a': 2740901422, 'p': '34066.18000000', 'q': '0.00082000', 'f': 3260167982, 'l': 3260167982, 'T': 1698479551490, 'm': True, 'M': True}
2023-10-28 13:26:01,991 - ERROR - Exception Occured ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))
2023-10-28 13:26:01,993 - ERROR - No data available
2023-10-28 13:26:06,059 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459044ED0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:06,060 - ERROR - No data available
2023-10-28 13:26:10,125 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590471D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:10,125 - ERROR - No data available
2023-10-28 13:26:14,191 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055190>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:14,192 - ERROR - No data available
2023-10-28 13:26:18,261 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057110>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:18,261 - ERROR - No data available
2023-10-28 13:26:22,350 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057210>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:22,350 - ERROR - No data available
2023-10-28 13:26:26,401 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459054790>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:26,404 - ERROR - No data available
2023-10-28 13:26:30,478 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047B90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:30,479 - ERROR - No data available
2023-10-28 13:26:34,555 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459045850>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:34,556 - ERROR - No data available
2023-10-28 13:26:38,616 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459033710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:38,619 - ERROR - No data available
2023-10-28 13:26:42,699 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905D290>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:42,701 - ERROR - No data available
2023-10-28 13:26:46,788 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590322D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:46,790 - ERROR - No data available
2023-10-28 13:26:50,847 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459044F90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:50,849 - ERROR - No data available
2023-10-28 13:26:54,913 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:54,914 - ERROR - No data available
2023-10-28 13:26:58,980 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023BD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:26:58,981 - ERROR - No data available
2023-10-28 13:27:03,039 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055AD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:03,042 - ERROR - No data available
2023-10-28 13:27:07,115 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905C790>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:07,117 - ERROR - No data available
2023-10-28 13:27:11,184 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:11,184 - ERROR - No data available
2023-10-28 13:27:15,251 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057C50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:15,252 - ERROR - No data available
2023-10-28 13:27:19,353 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055210>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:19,357 - ERROR - No data available
2023-10-28 13:27:23,409 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459044790>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:23,410 - ERROR - No data available
2023-10-28 13:27:27,475 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590458D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:27,475 - ERROR - No data available
2023-10-28 13:27:31,541 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E458DD9490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:31,543 - ERROR - No data available
2023-10-28 13:27:35,613 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E458DE5250>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:35,613 - ERROR - No data available
2023-10-28 13:27:39,715 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459045650>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:39,715 - ERROR - No data available
2023-10-28 13:27:43,772 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459045D50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:43,775 - ERROR - No data available
2023-10-28 13:27:47,856 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057B10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:47,858 - ERROR - No data available
2023-10-28 13:27:51,918 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055190>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:51,918 - ERROR - No data available
2023-10-28 13:27:55,969 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023010>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:27:55,972 - ERROR - No data available
2023-10-28 13:28:00,025 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905F490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:28:00,028 - ERROR - No data available
2023-10-28 13:28:04,104 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590569D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:28:04,104 - ERROR - No data available
2023-10-28 13:28:08,160 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056390>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:28:08,160 - ERROR - No data available
2023-10-28 13:28:12,209 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459033210>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:28:12,212 - ERROR - No data available
2023-10-28 13:28:20,850 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 13:28:20,850 - ERROR - No data available
2023-10-28 13:28:25,854 - INFO - {'e': 'aggTrade', 'E': 1698479894884, 's': 'BTCUSDT', 'a': 2740902752, 'p': '34052.13000000', 'q': '0.00303000', 'f': 3260169838, 'l': 3260169838, 'T': 1698479894883, 'm': True, 'M': True}
2023-10-28 13:28:25,854 - INFO - {'e': 'aggTrade', 'E': 1698479894884, 's': 'BTCUSDT', 'a': 2740902752, 'p': '34052.13000000', 'q': '0.00303000', 'f': 3260169838, 'l': 3260169838, 'T': 1698479894883, 'm': True, 'M': True}
2023-10-28 13:28:33,126 - INFO - {'e': 'aggTrade', 'E': 1698479894939, 's': 'BTCUSDT', 'a': 2740902753, 'p': '34052.13000000', 'q': '0.01000000', 'f': 3260169839, 'l': 3260169839, 'T': 1698479894938, 'm': True, 'M': True}
2023-10-28 13:28:33,126 - INFO - {'e': 'aggTrade', 'E': 1698479894939, 's': 'BTCUSDT', 'a': 2740902753, 'p': '34052.13000000', 'q': '0.01000000', 'f': 3260169839, 'l': 3260169839, 'T': 1698479894938, 'm': True, 'M': True}
2023-10-28 13:28:35,767 - INFO - {'e': 'aggTrade', 'E': 1698479895134, 's': 'BTCUSDT', 'a': 2740902754, 'p': '34052.13000000', 'q': '0.00120000', 'f': 3260169840, 'l': 3260169840, 'T': 1698479895134, 'm': True, 'M': True}
2023-10-28 13:28:35,767 - INFO - {'e': 'aggTrade', 'E': 1698479895134, 's': 'BTCUSDT', 'a': 2740902754, 'p': '34052.13000000', 'q': '0.00120000', 'f': 3260169840, 'l': 3260169840, 'T': 1698479895134, 'm': True, 'M': True}
2023-10-28 13:28:38,697 - INFO - {'e': 'aggTrade', 'E': 1698479895389, 's': 'BTCUSDT', 'a': 2740902755, 'p': '34052.13000000', 'q': '0.00468000', 'f': 3260169841, 'l': 3260169841, 'T': 1698479895388, 'm': True, 'M': True}
2023-10-28 13:28:38,697 - INFO - {'e': 'aggTrade', 'E': 1698479895389, 's': 'BTCUSDT', 'a': 2740902755, 'p': '34052.13000000', 'q': '0.00468000', 'f': 3260169841, 'l': 3260169841, 'T': 1698479895388, 'm': True, 'M': True}
2023-10-28 13:28:42,942 - INFO - {'e': 'aggTrade', 'E': 1698479896189, 's': 'BTCUSDT', 'a': 2740902756, 'p': '34052.13000000', 'q': '0.00200000', 'f': 3260169842, 'l': 3260169842, 'T': 1698479896189, 'm': True, 'M': True}
2023-10-28 13:28:42,943 - INFO - {'e': 'aggTrade', 'E': 1698479896189, 's': 'BTCUSDT', 'a': 2740902756, 'p': '34052.13000000', 'q': '0.00200000', 'f': 3260169842, 'l': 3260169842, 'T': 1698479896189, 'm': True, 'M': True}
2023-10-28 13:28:48,322 - INFO - {'e': 'aggTrade', 'E': 1698479898202, 's': 'BTCUSDT', 'a': 2740902759, 'p': '34052.13000000', 'q': '0.00346000', 'f': 3260169845, 'l': 3260169845, 'T': 1698479898202, 'm': True, 'M': True}
2023-10-28 13:28:48,324 - INFO - {'e': 'aggTrade', 'E': 1698479898202, 's': 'BTCUSDT', 'a': 2740902759, 'p': '34052.13000000', 'q': '0.00346000', 'f': 3260169845, 'l': 3260169845, 'T': 1698479898202, 'm': True, 'M': True}
2023-10-28 13:29:11,182 - INFO - {'e': 'aggTrade', 'E': 1698479898308, 's': 'BTCUSDT', 'a': 2740902761, 'p': '34052.13000000', 'q': '0.09068000', 'f': 3260169849, 'l': 3260169853, 'T': 1698479898307, 'm': True, 'M': True}
2023-10-28 13:29:11,183 - INFO - {'e': 'aggTrade', 'E': 1698479898308, 's': 'BTCUSDT', 'a': 2740902761, 'p': '34052.13000000', 'q': '0.09068000', 'f': 3260169849, 'l': 3260169853, 'T': 1698479898307, 'm': True, 'M': True}
2023-10-28 13:29:13,479 - INFO - {'e': 'aggTrade', 'E': 1698479898309, 's': 'BTCUSDT', 'a': 2740902763, 'p': '34052.13000000', 'q': '0.22533000', 'f': 3260169855, 'l': 3260169857, 'T': 1698479898308, 'm': True, 'M': True}
2023-10-28 13:29:13,481 - INFO - {'e': 'aggTrade', 'E': 1698479898309, 's': 'BTCUSDT', 'a': 2740902763, 'p': '34052.13000000', 'q': '0.22533000', 'f': 3260169855, 'l': 3260169857, 'T': 1698479898308, 'm': True, 'M': True}
2023-10-28 13:29:15,749 - INFO - {'e': 'aggTrade', 'E': 1698479898314, 's': 'BTCUSDT', 'a': 2740902764, 'p': '34051.99000000', 'q': '0.00100000', 'f': 3260169858, 'l': 3260169858, 'T': 1698479898312, 'm': True, 'M': True}
2023-10-28 13:29:15,753 - INFO - {'e': 'aggTrade', 'E': 1698479898314, 's': 'BTCUSDT', 'a': 2740902764, 'p': '34051.99000000', 'q': '0.00100000', 'f': 3260169858, 'l': 3260169858, 'T': 1698479898312, 'm': True, 'M': True}
2023-10-28 13:29:18,098 - INFO - {'e': 'aggTrade', 'E': 1698479898327, 's': 'BTCUSDT', 'a': 2740902767, 'p': '34051.54000000', 'q': '0.00293000', 'f': 3260169861, 'l': 3260169861, 'T': 1698479898326, 'm': True, 'M': True}
2023-10-28 13:29:18,098 - INFO - {'e': 'aggTrade', 'E': 1698479898327, 's': 'BTCUSDT', 'a': 2740902767, 'p': '34051.54000000', 'q': '0.00293000', 'f': 3260169861, 'l': 3260169861, 'T': 1698479898326, 'm': True, 'M': True}
2023-10-28 13:29:23,316 - INFO - {'e': 'aggTrade', 'E': 1698479898327, 's': 'BTCUSDT', 'a': 2740902769, 'p': '34051.52000000', 'q': '0.00129000', 'f': 3260169869, 'l': 3260169869, 'T': 1698479898326, 'm': True, 'M': True}
2023-10-28 13:29:23,317 - INFO - {'e': 'aggTrade', 'E': 1698479898327, 's': 'BTCUSDT', 'a': 2740902769, 'p': '34051.52000000', 'q': '0.00129000', 'f': 3260169869, 'l': 3260169869, 'T': 1698479898326, 'm': True, 'M': True}
2023-10-28 13:29:25,630 - INFO - {'e': 'aggTrade', 'E': 1698479898334, 's': 'BTCUSDT', 'a': 2740902772, 'p': '34051.19000000', 'q': '0.00500000', 'f': 3260169872, 'l': 3260169872, 'T': 1698479898333, 'm': True, 'M': True}
2023-10-28 13:29:25,632 - INFO - {'e': 'aggTrade', 'E': 1698479898334, 's': 'BTCUSDT', 'a': 2740902772, 'p': '34051.19000000', 'q': '0.00500000', 'f': 3260169872, 'l': 3260169872, 'T': 1698479898333, 'm': True, 'M': True}
2023-10-28 13:29:29,704 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459044990>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:29:29,708 - ERROR - No data available
2023-10-28 13:29:33,753 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590476D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:29:33,753 - ERROR - No data available
2023-10-28 13:29:37,819 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905CCD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:29:37,819 - ERROR - No data available
2023-10-28 13:29:41,892 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459032F10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:29:41,897 - ERROR - No data available
2023-10-28 13:29:45,965 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047250>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:29:45,965 - ERROR - No data available
2023-10-28 13:29:50,035 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590541D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:29:50,035 - ERROR - No data available
2023-10-28 13:29:54,096 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590555D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:29:54,107 - ERROR - No data available
2023-10-28 13:29:58,157 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:29:58,159 - ERROR - No data available
2023-10-28 13:30:02,230 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905C690>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:02,230 - ERROR - No data available
2023-10-28 13:30:06,313 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055650>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:06,313 - ERROR - No data available
2023-10-28 13:30:10,364 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057390>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:10,367 - ERROR - No data available
2023-10-28 13:30:14,425 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047A50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:14,428 - ERROR - No data available
2023-10-28 13:30:18,497 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459046A90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:18,497 - ERROR - No data available
2023-10-28 13:30:22,570 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4588FACD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:22,573 - ERROR - No data available
2023-10-28 13:30:26,625 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45906D390>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:26,625 - ERROR - No data available
2023-10-28 13:30:30,700 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590320D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:30,700 - ERROR - No data available
2023-10-28 13:30:34,761 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459046990>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:34,761 - ERROR - No data available
2023-10-28 13:30:38,819 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055A90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:38,821 - ERROR - No data available
2023-10-28 13:30:42,889 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:42,890 - ERROR - No data available
2023-10-28 13:30:47,003 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023150>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:47,003 - ERROR - No data available
2023-10-28 13:30:51,081 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45906F350>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:51,081 - ERROR - No data available
2023-10-28 13:30:55,134 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459032910>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:55,134 - ERROR - No data available
2023-10-28 13:30:59,209 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056250>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:30:59,209 - ERROR - No data available
2023-10-28 13:31:03,281 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055690>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:03,281 - ERROR - No data available
2023-10-28 13:31:07,345 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905CDD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:07,345 - ERROR - No data available
2023-10-28 13:31:11,406 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905C0D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:11,406 - ERROR - No data available
2023-10-28 13:31:15,486 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459046490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:15,486 - ERROR - No data available
2023-10-28 13:31:19,545 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056E10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:19,545 - ERROR - No data available
2023-10-28 13:31:23,603 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:23,603 - ERROR - No data available
2023-10-28 13:31:27,677 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023B50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:27,677 - ERROR - No data available
2023-10-28 13:31:31,761 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E0D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:31,761 - ERROR - No data available
2023-10-28 13:31:35,833 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905C910>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:35,833 - ERROR - No data available
2023-10-28 13:31:39,897 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590451D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:39,897 - ERROR - No data available
2023-10-28 13:31:43,970 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590321D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:43,970 - ERROR - No data available
2023-10-28 13:31:48,035 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E6D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:48,035 - ERROR - No data available
2023-10-28 13:31:52,124 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905CBD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:52,124 - ERROR - No data available
2023-10-28 13:31:56,191 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459054350>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:31:56,192 - ERROR - No data available
2023-10-28 13:32:00,256 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459045790>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:00,256 - ERROR - No data available
2023-10-28 13:32:04,328 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45906D010>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:04,328 - ERROR - No data available
2023-10-28 13:32:08,381 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056ED0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:08,384 - ERROR - No data available
2023-10-28 13:32:12,450 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056C90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:12,452 - ERROR - No data available
2023-10-28 13:32:16,527 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905F690>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:16,527 - ERROR - No data available
2023-10-28 13:32:20,602 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4567B7050>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:20,602 - ERROR - No data available
2023-10-28 13:32:24,665 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459044490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:24,667 - ERROR - No data available
2023-10-28 13:32:28,734 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459075310>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:28,734 - ERROR - No data available
2023-10-28 13:32:32,788 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459031FD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:32,788 - ERROR - No data available
2023-10-28 13:32:36,858 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023A90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:36,858 - ERROR - No data available
2023-10-28 13:32:40,911 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E0D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:40,911 - ERROR - No data available
2023-10-28 13:32:44,971 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055C10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:44,971 - ERROR - No data available
2023-10-28 13:32:49,061 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590554D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:49,061 - ERROR - No data available
2023-10-28 13:32:53,100 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590772D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:53,103 - ERROR - No data available
2023-10-28 13:32:57,175 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459054F10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:32:57,177 - ERROR - No data available
2023-10-28 13:33:01,247 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056350>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:01,247 - ERROR - No data available
2023-10-28 13:33:05,324 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905C090>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:05,324 - ERROR - No data available
2023-10-28 13:33:09,368 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023050>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:09,370 - ERROR - No data available
2023-10-28 13:33:13,453 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E458FB4810>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:13,453 - ERROR - No data available
2023-10-28 13:33:17,527 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590812D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:17,527 - ERROR - No data available
2023-10-28 13:33:21,608 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E458FA68D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:21,609 - ERROR - No data available
2023-10-28 13:33:25,682 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E456F2E7D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:25,682 - ERROR - No data available
2023-10-28 13:33:29,753 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905D050>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:29,754 - ERROR - No data available
2023-10-28 13:33:33,821 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590540D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:33,821 - ERROR - No data available
2023-10-28 13:33:37,857 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055750>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:37,859 - ERROR - No data available
2023-10-28 13:33:41,916 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459083250>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:41,916 - ERROR - No data available
2023-10-28 13:33:45,986 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055E90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:45,987 - ERROR - No data available
2023-10-28 13:33:50,061 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459054CD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:50,061 - ERROR - No data available
2023-10-28 13:33:54,137 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905CB50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:54,137 - ERROR - No data available
2023-10-28 13:33:58,247 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023690>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:33:58,247 - ERROR - No data available
2023-10-28 13:34:02,319 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4588FA050>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:02,320 - ERROR - No data available
2023-10-28 13:34:06,372 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459085250>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:06,372 - ERROR - No data available
2023-10-28 13:34:10,430 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047A50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:10,430 - ERROR - No data available
2023-10-28 13:34:14,512 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905CE90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:14,512 - ERROR - No data available
2023-10-28 13:34:18,594 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905D5D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:18,596 - ERROR - No data available
2023-10-28 13:34:22,640 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057510>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:22,640 - ERROR - No data available
2023-10-28 13:34:26,705 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590552D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:26,705 - ERROR - No data available
2023-10-28 13:34:30,771 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459087210>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:30,771 - ERROR - No data available
2023-10-28 13:34:34,813 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:34,813 - ERROR - No data available
2023-10-28 13:34:38,918 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45906C110>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:38,918 - ERROR - No data available
2023-10-28 13:34:42,973 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905CF10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:42,973 - ERROR - No data available
2023-10-28 13:34:47,035 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023310>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:47,035 - ERROR - No data available
2023-10-28 13:34:51,095 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E458FB4810>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:51,095 - ERROR - No data available
2023-10-28 13:34:55,183 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908D210>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:55,186 - ERROR - No data available
2023-10-28 13:34:59,247 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459022E50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:34:59,248 - ERROR - No data available
2023-10-28 13:35:03,312 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905D650>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:03,312 - ERROR - No data available
2023-10-28 13:35:07,379 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459080890>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:07,379 - ERROR - No data available
2023-10-28 13:35:11,438 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057B50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:11,438 - ERROR - No data available
2023-10-28 13:35:15,502 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459054210>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:15,502 - ERROR - No data available
2023-10-28 13:35:19,594 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908F210>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:19,594 - ERROR - No data available
2023-10-28 13:35:23,687 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590540D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:23,687 - ERROR - No data available
2023-10-28 13:35:27,765 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45906FD90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:27,765 - ERROR - No data available
2023-10-28 13:35:31,838 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E150>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:31,838 - ERROR - No data available
2023-10-28 13:35:35,925 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459032550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:35,925 - ERROR - No data available
2023-10-28 13:35:39,998 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908E0D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:40,000 - ERROR - No data available
2023-10-28 13:35:44,082 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459095250>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:44,082 - ERROR - No data available
2023-10-28 13:35:48,172 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908F3D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:48,172 - ERROR - No data available
2023-10-28 13:35:52,251 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908CDD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:52,253 - ERROR - No data available
2023-10-28 13:35:56,309 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459087ED0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:35:56,309 - ERROR - No data available
2023-10-28 13:36:00,382 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459085D90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:00,382 - ERROR - No data available
2023-10-28 13:36:04,485 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459022D50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:04,485 - ERROR - No data available
2023-10-28 13:36:08,553 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590830D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:08,553 - ERROR - No data available
2023-10-28 13:36:12,620 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590850D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:12,620 - ERROR - No data available
2023-10-28 13:36:16,676 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459085FD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:16,676 - ERROR - No data available
2023-10-28 13:36:20,753 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459085850>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:20,753 - ERROR - No data available
2023-10-28 13:36:24,831 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908DC10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:24,831 - ERROR - No data available
2023-10-28 13:36:28,917 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459083AD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:28,917 - ERROR - No data available
2023-10-28 13:36:32,995 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459080090>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:32,995 - ERROR - No data available
2023-10-28 13:36:37,067 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908D1D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:37,067 - ERROR - No data available
2023-10-28 13:36:41,128 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908DB10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:41,128 - ERROR - No data available
2023-10-28 13:36:45,212 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459087F90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:45,212 - ERROR - No data available
2023-10-28 13:36:49,302 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459086250>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:49,302 - ERROR - No data available
2023-10-28 13:36:53,361 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590837D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:53,361 - ERROR - No data available
2023-10-28 13:36:57,434 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459076F90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:36:57,434 - ERROR - No data available
2023-10-28 13:37:01,503 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459084290>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:01,503 - ERROR - No data available
2023-10-28 13:37:05,585 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459086290>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:05,585 - ERROR - No data available
2023-10-28 13:37:09,639 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E457081B90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:09,639 - ERROR - No data available
2023-10-28 13:37:13,706 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908E810>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:13,706 - ERROR - No data available
2023-10-28 13:37:17,761 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E458FA68D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:17,761 - ERROR - No data available
2023-10-28 13:37:21,818 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459074210>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:21,818 - ERROR - No data available
2023-10-28 13:37:25,866 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908F6D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:25,866 - ERROR - No data available
2023-10-28 13:37:29,935 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908CF10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:29,937 - ERROR - No data available
2023-10-28 13:37:34,008 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459087750>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:34,008 - ERROR - No data available
2023-10-28 13:37:38,073 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590852D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:38,075 - ERROR - No data available
2023-10-28 13:37:42,154 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459076250>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:42,154 - ERROR - No data available
2023-10-28 13:37:46,218 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45906EA50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:46,218 - ERROR - No data available
2023-10-28 13:37:50,291 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459087550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:50,293 - ERROR - No data available
2023-10-28 13:37:54,368 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459085D90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:54,368 - ERROR - No data available
2023-10-28 13:37:58,443 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459083A90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:37:58,445 - ERROR - No data available
2023-10-28 13:38:02,509 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908DC90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:02,509 - ERROR - No data available
2023-10-28 13:38:06,625 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E458FB4810>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:06,625 - ERROR - No data available
2023-10-28 13:38:10,718 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056950>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:10,719 - ERROR - No data available
2023-10-28 13:38:14,790 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908F6D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:14,793 - ERROR - No data available
2023-10-28 13:38:18,851 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908CD10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:18,852 - ERROR - No data available
2023-10-28 13:38:22,902 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459084350>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:22,902 - ERROR - No data available
2023-10-28 13:38:27,009 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459084250>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:27,009 - ERROR - No data available
2023-10-28 13:38:31,080 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023190>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:31,082 - ERROR - No data available
2023-10-28 13:38:35,146 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055E90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:35,146 - ERROR - No data available
2023-10-28 13:38:39,206 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590841D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:39,206 - ERROR - No data available
2023-10-28 13:38:43,260 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459085650>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:43,261 - ERROR - No data available
2023-10-28 13:38:47,330 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459081750>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:47,330 - ERROR - No data available
2023-10-28 13:38:51,412 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908C810>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:51,415 - ERROR - No data available
2023-10-28 13:38:55,480 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E458958690>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:55,482 - ERROR - No data available
2023-10-28 13:38:59,566 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459046710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:38:59,566 - ERROR - No data available
2023-10-28 13:39:03,665 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908EC90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:03,666 - ERROR - No data available
2023-10-28 13:39:07,740 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45906D310>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:07,741 - ERROR - No data available
2023-10-28 13:39:11,806 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459086110>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:11,809 - ERROR - No data available
2023-10-28 13:39:15,867 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459087AD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:15,869 - ERROR - No data available
2023-10-28 13:39:19,907 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056810>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:19,907 - ERROR - No data available
2023-10-28 13:39:23,947 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047C90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:23,947 - ERROR - No data available
2023-10-28 13:39:28,017 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590863D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:28,021 - ERROR - No data available
2023-10-28 13:39:32,079 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590871D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:32,080 - ERROR - No data available
2023-10-28 13:39:36,133 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459082B90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:36,135 - ERROR - No data available
2023-10-28 13:39:40,187 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908EA50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:40,189 - ERROR - No data available
2023-10-28 13:39:44,214 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E456912F10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:44,215 - ERROR - No data available
2023-10-28 13:39:48,266 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905D350>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:48,266 - ERROR - No data available
2023-10-28 13:39:52,311 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908F7D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:52,313 - ERROR - No data available
2023-10-28 13:39:56,377 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459077390>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:39:56,377 - ERROR - No data available
2023-10-28 13:40:00,459 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459033410>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:00,460 - ERROR - No data available
2023-10-28 13:40:04,505 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459084550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:04,505 - ERROR - No data available
2023-10-28 13:40:08,571 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459023190>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:08,571 - ERROR - No data available
2023-10-28 13:40:12,642 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905DF50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:12,645 - ERROR - No data available
2023-10-28 13:40:16,677 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590863D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:16,677 - ERROR - No data available
2023-10-28 13:40:20,758 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459085350>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:20,759 - ERROR - No data available
2023-10-28 13:40:24,802 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459081FD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:24,803 - ERROR - No data available
2023-10-28 13:40:28,855 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908D3D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:28,857 - ERROR - No data available
2023-10-28 13:40:32,913 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E090>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:32,914 - ERROR - No data available
2023-10-28 13:40:36,993 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459097050>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:36,993 - ERROR - No data available
2023-10-28 13:40:41,068 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E5D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:41,071 - ERROR - No data available
2023-10-28 13:40:45,118 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E950>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:45,119 - ERROR - No data available
2023-10-28 13:40:49,166 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047F10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:49,166 - ERROR - No data available
2023-10-28 13:40:53,230 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047E10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:53,230 - ERROR - No data available
2023-10-28 13:40:57,294 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057650>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:40:57,294 - ERROR - No data available
2023-10-28 13:41:01,363 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590557D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:01,363 - ERROR - No data available
2023-10-28 13:41:05,411 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459045550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:05,413 - ERROR - No data available
2023-10-28 13:41:09,478 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590455D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:09,478 - ERROR - No data available
2023-10-28 13:41:13,561 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905C1D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:13,561 - ERROR - No data available
2023-10-28 13:41:17,630 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:17,630 - ERROR - No data available
2023-10-28 13:41:21,710 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459054E50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:21,712 - ERROR - No data available
2023-10-28 13:41:25,774 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45906C950>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:25,774 - ERROR - No data available
2023-10-28 13:41:29,824 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905EF10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:29,826 - ERROR - No data available
2023-10-28 13:41:33,889 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:33,891 - ERROR - No data available
2023-10-28 13:41:37,936 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056B90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:37,938 - ERROR - No data available
2023-10-28 13:41:41,986 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459045B10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:41,988 - ERROR - No data available
2023-10-28 13:41:46,048 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459046350>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:46,051 - ERROR - No data available
2023-10-28 13:41:50,107 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459032450>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:50,108 - ERROR - No data available
2023-10-28 13:41:54,166 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459044BD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:54,167 - ERROR - No data available
2023-10-28 13:41:58,225 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459046D10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:41:58,225 - ERROR - No data available
2023-10-28 13:42:02,314 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905F0D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:02,315 - ERROR - No data available
2023-10-28 13:42:06,432 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905FCD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:06,432 - ERROR - No data available
2023-10-28 13:42:10,501 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45906D0D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:10,501 - ERROR - No data available
2023-10-28 13:42:14,567 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459074BD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:14,567 - ERROR - No data available
2023-10-28 13:42:18,639 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905EFD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:18,641 - ERROR - No data available
2023-10-28 13:42:22,693 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905F910>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:22,696 - ERROR - No data available
2023-10-28 13:42:26,757 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057810>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:26,758 - ERROR - No data available
2023-10-28 13:42:30,813 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047610>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:30,815 - ERROR - No data available
2023-10-28 13:42:34,874 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047B10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:34,876 - ERROR - No data available
2023-10-28 13:42:38,925 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459077290>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:38,928 - ERROR - No data available
2023-10-28 13:42:42,990 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459046710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:42,992 - ERROR - No data available
2023-10-28 13:42:47,038 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459045010>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:47,040 - ERROR - No data available
2023-10-28 13:42:51,107 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905F290>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:51,109 - ERROR - No data available
2023-10-28 13:42:55,170 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905FF90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:55,172 - ERROR - No data available
2023-10-28 13:42:59,218 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459077C50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:42:59,221 - ERROR - No data available
2023-10-28 13:43:03,252 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908D010>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:03,252 - ERROR - No data available
2023-10-28 13:43:07,315 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905DC50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:07,317 - ERROR - No data available
2023-10-28 13:43:11,383 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905D950>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:11,386 - ERROR - No data available
2023-10-28 13:43:15,445 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056C50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:15,447 - ERROR - No data available
2023-10-28 13:43:19,496 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459046510>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:19,500 - ERROR - No data available
2023-10-28 13:43:23,544 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047A50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:23,547 - ERROR - No data available
2023-10-28 13:43:27,594 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459081990>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:27,594 - ERROR - No data available
2023-10-28 13:43:31,645 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459044550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:31,647 - ERROR - No data available
2023-10-28 13:43:35,693 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459077F10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:35,696 - ERROR - No data available
2023-10-28 13:43:39,738 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459057390>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:39,739 - ERROR - No data available
2023-10-28 13:43:43,775 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E4D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:43,778 - ERROR - No data available
2023-10-28 13:43:47,821 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908DF10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:47,824 - ERROR - No data available
2023-10-28 13:43:51,872 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459083BD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:51,873 - ERROR - No data available
2023-10-28 13:43:55,916 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905C690>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:55,916 - ERROR - No data available
2023-10-28 13:43:59,965 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905CAD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:43:59,967 - ERROR - No data available
2023-10-28 13:44:04,020 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459056C90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:04,022 - ERROR - No data available
2023-10-28 13:44:08,069 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4590475D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:08,072 - ERROR - No data available
2023-10-28 13:44:12,100 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459045E90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:12,103 - ERROR - No data available
2023-10-28 13:44:16,154 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459086C10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:16,154 - ERROR - No data available
2023-10-28 13:44:20,233 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459047F10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:20,236 - ERROR - No data available
2023-10-28 13:44:24,294 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45908D690>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:24,296 - ERROR - No data available
2023-10-28 13:44:28,364 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459055B10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:28,366 - ERROR - No data available
2023-10-28 13:44:32,415 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905F350>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:32,418 - ERROR - No data available
2023-10-28 13:44:36,458 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459081210>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:36,459 - ERROR - No data available
2023-10-28 13:44:40,510 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E459087A90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:40,510 - ERROR - No data available
2023-10-28 13:44:44,613 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905E1D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:44,613 - ERROR - No data available
2023-10-28 13:44:48,692 - ERROR - Exception Occured HTTPConnectionPool(host='localhost', port=5000): Max retries exceeded with url: /ltm (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E45905F250>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2023-10-28 13:44:48,693 - ERROR - No data available
2023-10-28 14:14:35,134 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 14:14:36,754 - ERROR - No data available
2023-10-28 16:45:06,220 - INFO - {'e': 'aggTrade', 'E': 1698491707217, 's': 'BTCUSDT', 'a': 2740955335, 'p': '34165.98000000', 'q': '0.00030000', 'f': 3260237253, 'l': 3260237253, 'T': 1698491707216, 'm': True, 'M': True}
2023-10-28 16:45:06,220 - INFO - {'e': 'aggTrade', 'E': 1698491707217, 's': 'BTCUSDT', 'a': 2740955335, 'p': '34165.98000000', 'q': '0.00030000', 'f': 3260237253, 'l': 3260237253, 'T': 1698491707216, 'm': True, 'M': True}
2023-10-28 16:45:08,263 - INFO - {'e': 'aggTrade', 'E': 1698491709149, 's': 'ETHUSDT', 'a': 949460899, 'p': '1794.39000000', 'q': '0.00560000', 'f': 1218941204, 'l': 1218941204, 'T': 1698491709148, 'm': False, 'M': True}
2023-10-28 16:45:08,263 - INFO - {'e': 'aggTrade', 'E': 1698491709149, 's': 'ETHUSDT', 'a': 949460899, 'p': '1794.39000000', 'q': '0.00560000', 'f': 1218941204, 'l': 1218941204, 'T': 1698491709148, 'm': False, 'M': True}
2023-10-28 16:45:10,296 - INFO - {'e': 'aggTrade', 'E': 1698491711088, 's': 'ETHUSDT', 'a': 949460900, 'p': '1794.41000000', 'q': '0.01660000', 'f': 1218941205, 'l': 1218941205, 'T': 1698491711087, 'm': False, 'M': True}
2023-10-28 16:45:10,296 - INFO - {'e': 'aggTrade', 'E': 1698491711088, 's': 'ETHUSDT', 'a': 949460900, 'p': '1794.41000000', 'q': '0.01660000', 'f': 1218941205, 'l': 1218941205, 'T': 1698491711087, 'm': False, 'M': True}
2023-10-28 16:45:12,323 - INFO - {'e': 'aggTrade', 'E': 1698491713399, 's': 'BTCUSDT', 'a': 2740955343, 'p': '34165.99000000', 'q': '0.00028000', 'f': 3260237261, 'l': 3260237261, 'T': 1698491713398, 'm': False, 'M': True}
2023-10-28 16:45:12,325 - INFO - {'e': 'aggTrade', 'E': 1698491713399, 's': 'BTCUSDT', 'a': 2740955343, 'p': '34165.99000000', 'q': '0.00028000', 'f': 3260237261, 'l': 3260237261, 'T': 1698491713398, 'm': False, 'M': True}
2023-10-28 16:45:14,343 - INFO - {'e': 'aggTrade', 'E': 1698491715425, 's': 'BTCUSDT', 'a': 2740955345, 'p': '34165.99000000', 'q': '0.03745000', 'f': 3260237263, 'l': 3260237263, 'T': 1698491715424, 'm': False, 'M': True}
2023-10-28 16:45:14,345 - INFO - {'e': 'aggTrade', 'E': 1698491715425, 's': 'BTCUSDT', 'a': 2740955345, 'p': '34165.99000000', 'q': '0.03745000', 'f': 3260237263, 'l': 3260237263, 'T': 1698491715424, 'm': False, 'M': True}
2023-10-28 16:45:16,387 - INFO - {'e': 'aggTrade', 'E': 1698491717425, 's': 'BTCUSDT', 'a': 2740955349, 'p': '34165.99000000', 'q': '0.04999000', 'f': 3260237267, 'l': 3260237267, 'T': 1698491717424, 'm': False, 'M': True}
2023-10-28 16:45:16,387 - INFO - {'e': 'aggTrade', 'E': 1698491717425, 's': 'BTCUSDT', 'a': 2740955349, 'p': '34165.99000000', 'q': '0.04999000', 'f': 3260237267, 'l': 3260237267, 'T': 1698491717424, 'm': False, 'M': True}
2023-10-28 16:45:18,450 - INFO - {'e': 'aggTrade', 'E': 1698491719245, 's': 'BTCUSDT', 'a': 2740955356, 'p': '34165.98000000', 'q': '0.01200000', 'f': 3260237277, 'l': 3260237277, 'T': 1698491719245, 'm': True, 'M': True}
2023-10-28 16:45:18,450 - INFO - {'e': 'aggTrade', 'E': 1698491719245, 's': 'BTCUSDT', 'a': 2740955356, 'p': '34165.98000000', 'q': '0.01200000', 'f': 3260237277, 'l': 3260237277, 'T': 1698491719245, 'm': True, 'M': True}
2023-10-28 16:45:20,491 - INFO - {'e': 'aggTrade', 'E': 1698491721576, 's': 'ETHUSDT', 'a': 949460911, 'p': '1794.41000000', 'q': '0.99460000', 'f': 1218941217, 'l': 1218941217, 'T': 1698491721576, 'm': False, 'M': True}
2023-10-28 16:45:20,494 - INFO - {'e': 'aggTrade', 'E': 1698491721576, 's': 'ETHUSDT', 'a': 949460911, 'p': '1794.41000000', 'q': '0.99460000', 'f': 1218941217, 'l': 1218941217, 'T': 1698491721576, 'm': False, 'M': True}
2023-10-28 16:45:22,560 - INFO - {'e': 'aggTrade', 'E': 1698491723444, 's': 'BTCUSDT', 'a': 2740955379, 'p': '34165.99000000', 'q': '0.04999000', 'f': 3260237303, 'l': 3260237303, 'T': 1698491723444, 'm': False, 'M': True}
2023-10-28 16:45:22,564 - INFO - {'e': 'aggTrade', 'E': 1698491723444, 's': 'BTCUSDT', 'a': 2740955379, 'p': '34165.99000000', 'q': '0.04999000', 'f': 3260237303, 'l': 3260237303, 'T': 1698491723444, 'm': False, 'M': True}
2023-10-28 16:45:24,597 - INFO - {'e': 'aggTrade', 'E': 1698491725446, 's': 'BTCUSDT', 'a': 2740955387, 'p': '34165.99000000', 'q': '0.03504000', 'f': 3260237321, 'l': 3260237322, 'T': 1698491725445, 'm': False, 'M': True}
2023-10-28 16:45:24,609 - INFO - {'e': 'aggTrade', 'E': 1698491725446, 's': 'BTCUSDT', 'a': 2740955387, 'p': '34165.99000000', 'q': '0.03504000', 'f': 3260237321, 'l': 3260237322, 'T': 1698491725445, 'm': False, 'M': True}
2023-10-28 16:45:26,665 - INFO - {'e': 'aggTrade', 'E': 1698491727725, 's': 'BTCUSDT', 'a': 2740955403, 'p': '34167.87000000', 'q': '0.00018000', 'f': 3260237340, 'l': 3260237340, 'T': 1698491727725, 'm': True, 'M': True}
2023-10-28 16:45:26,667 - INFO - {'e': 'aggTrade', 'E': 1698491727725, 's': 'BTCUSDT', 'a': 2740955403, 'p': '34167.87000000', 'q': '0.00018000', 'f': 3260237340, 'l': 3260237340, 'T': 1698491727725, 'm': True, 'M': True}
2023-10-28 16:45:28,699 - INFO - {'e': 'aggTrade', 'E': 1698491729825, 's': 'ETHUSDT', 'a': 949460914, 'p': '1794.41000000', 'q': '0.09950000', 'f': 1218941220, 'l': 1218941221, 'T': 1698491729825, 'm': False, 'M': True}
2023-10-28 16:45:28,700 - INFO - {'e': 'aggTrade', 'E': 1698491729825, 's': 'ETHUSDT', 'a': 949460914, 'p': '1794.41000000', 'q': '0.09950000', 'f': 1218941220, 'l': 1218941221, 'T': 1698491729825, 'm': False, 'M': True}
2023-10-28 16:45:30,739 - INFO - {'e': 'aggTrade', 'E': 1698491731458, 's': 'BTCUSDT', 'a': 2740955431, 'p': '34169.99000000', 'q': '0.04999000', 'f': 3260237373, 'l': 3260237373, 'T': 1698491731457, 'm': False, 'M': True}
2023-10-28 16:45:30,739 - INFO - {'e': 'aggTrade', 'E': 1698491731458, 's': 'BTCUSDT', 'a': 2740955431, 'p': '34169.99000000', 'q': '0.04999000', 'f': 3260237373, 'l': 3260237373, 'T': 1698491731457, 'm': False, 'M': True}
2023-10-28 16:45:32,804 - INFO - {'e': 'aggTrade', 'E': 1698491733699, 's': 'BTCUSDT', 'a': 2740955441, 'p': '34169.99000000', 'q': '0.00038000', 'f': 3260237387, 'l': 3260237387, 'T': 1698491733698, 'm': True, 'M': True}
2023-10-28 16:45:32,804 - INFO - {'e': 'aggTrade', 'E': 1698491733699, 's': 'BTCUSDT', 'a': 2740955441, 'p': '34169.99000000', 'q': '0.00038000', 'f': 3260237387, 'l': 3260237387, 'T': 1698491733698, 'm': True, 'M': True}
2023-10-28 16:45:34,849 - INFO - {'e': 'aggTrade', 'E': 1698491735489, 's': 'BTCUSDT', 'a': 2740955448, 'p': '34170.00000000', 'q': '0.04999000', 'f': 3260237394, 'l': 3260237394, 'T': 1698491735489, 'm': False, 'M': True}
2023-10-28 16:45:34,849 - INFO - {'e': 'aggTrade', 'E': 1698491735489, 's': 'BTCUSDT', 'a': 2740955448, 'p': '34170.00000000', 'q': '0.04999000', 'f': 3260237394, 'l': 3260237394, 'T': 1698491735489, 'm': False, 'M': True}
2023-10-28 16:45:36,907 - INFO - {'e': 'aggTrade', 'E': 1698491737839, 's': 'BTCUSDT', 'a': 2740955454, 'p': '34170.00000000', 'q': '0.00267000', 'f': 3260237400, 'l': 3260237400, 'T': 1698491737839, 'm': False, 'M': True}
2023-10-28 16:45:36,907 - INFO - {'e': 'aggTrade', 'E': 1698491737839, 's': 'BTCUSDT', 'a': 2740955454, 'p': '34170.00000000', 'q': '0.00267000', 'f': 3260237400, 'l': 3260237400, 'T': 1698491737839, 'm': False, 'M': True}
2023-10-28 16:45:38,961 - INFO - {'e': 'aggTrade', 'E': 1698491740001, 's': 'BTCUSDT', 'a': 2740955459, 'p': '34169.99000000', 'q': '0.00063000', 'f': 3260237405, 'l': 3260237405, 'T': 1698491740000, 'm': True, 'M': True}
2023-10-28 16:45:38,961 - INFO - {'e': 'aggTrade', 'E': 1698491740001, 's': 'BTCUSDT', 'a': 2740955459, 'p': '34169.99000000', 'q': '0.00063000', 'f': 3260237405, 'l': 3260237405, 'T': 1698491740000, 'm': True, 'M': True}
2023-10-28 16:45:41,000 - INFO - {'e': 'aggTrade', 'E': 1698491741984, 's': 'BTCUSDT', 'a': 2740955460, 'p': '34169.99000000', 'q': '0.00086000', 'f': 3260237406, 'l': 3260237406, 'T': 1698491741983, 'm': True, 'M': True}
2023-10-28 16:45:41,003 - INFO - {'e': 'aggTrade', 'E': 1698491741984, 's': 'BTCUSDT', 'a': 2740955460, 'p': '34169.99000000', 'q': '0.00086000', 'f': 3260237406, 'l': 3260237406, 'T': 1698491741983, 'm': True, 'M': True}
2023-10-28 16:45:43,051 - INFO - {'e': 'aggTrade', 'E': 1698491744195, 's': 'BTCUSDT', 'a': 2740955465, 'p': '34170.00000000', 'q': '0.00089000', 'f': 3260237411, 'l': 3260237411, 'T': 1698491744194, 'm': False, 'M': True}
2023-10-28 16:45:43,051 - INFO - {'e': 'aggTrade', 'E': 1698491744195, 's': 'BTCUSDT', 'a': 2740955465, 'p': '34170.00000000', 'q': '0.00089000', 'f': 3260237411, 'l': 3260237411, 'T': 1698491744194, 'm': False, 'M': True}
2023-10-28 16:45:45,090 - INFO - {'e': 'aggTrade', 'E': 1698491745891, 's': 'BTCUSDT', 'a': 2740955470, 'p': '34169.99000000', 'q': '0.00697000', 'f': 3260237416, 'l': 3260237416, 'T': 1698491745891, 'm': True, 'M': True}
2023-10-28 16:45:45,090 - INFO - {'e': 'aggTrade', 'E': 1698491745891, 's': 'BTCUSDT', 'a': 2740955470, 'p': '34169.99000000', 'q': '0.00697000', 'f': 3260237416, 'l': 3260237416, 'T': 1698491745891, 'm': True, 'M': True}
2023-10-28 16:45:47,132 - INFO - {'e': 'aggTrade', 'E': 1698491747887, 's': 'ETHUSDT', 'a': 949460947, 'p': '1794.72000000', 'q': '2.13410000', 'f': 1218941284, 'l': 1218941284, 'T': 1698491747886, 'm': False, 'M': True}
2023-10-28 16:45:47,132 - INFO - {'e': 'aggTrade', 'E': 1698491747887, 's': 'ETHUSDT', 'a': 949460947, 'p': '1794.72000000', 'q': '2.13410000', 'f': 1218941284, 'l': 1218941284, 'T': 1698491747886, 'm': False, 'M': True}
2023-10-28 16:45:49,176 - INFO - {'e': 'aggTrade', 'E': 1698491750070, 's': 'BTCUSDT', 'a': 2740955477, 'p': '34169.99000000', 'q': '0.00038000', 'f': 3260237423, 'l': 3260237423, 'T': 1698491750069, 'm': True, 'M': True}
2023-10-28 16:45:49,176 - INFO - {'e': 'aggTrade', 'E': 1698491750070, 's': 'BTCUSDT', 'a': 2740955477, 'p': '34169.99000000', 'q': '0.00038000', 'f': 3260237423, 'l': 3260237423, 'T': 1698491750069, 'm': True, 'M': True}
2023-10-28 16:45:51,239 - INFO - {'e': 'aggTrade', 'E': 1698491751497, 's': 'ETHUSDT', 'a': 949460951, 'p': '1794.72000000', 'q': '0.11030000', 'f': 1218941288, 'l': 1218941288, 'T': 1698491751497, 'm': False, 'M': True}
2023-10-28 16:45:51,241 - INFO - {'e': 'aggTrade', 'E': 1698491751497, 's': 'ETHUSDT', 'a': 949460951, 'p': '1794.72000000', 'q': '0.11030000', 'f': 1218941288, 'l': 1218941288, 'T': 1698491751497, 'm': False, 'M': True}
2023-10-28 16:45:53,317 - INFO - {'e': 'aggTrade', 'E': 1698491754452, 's': 'BTCUSDT', 'a': 2740955484, 'p': '34169.99000000', 'q': '0.00268000', 'f': 3260237430, 'l': 3260237430, 'T': 1698491754451, 'm': True, 'M': True}
2023-10-28 16:45:53,321 - INFO - {'e': 'aggTrade', 'E': 1698491754452, 's': 'BTCUSDT', 'a': 2740955484, 'p': '34169.99000000', 'q': '0.00268000', 'f': 3260237430, 'l': 3260237430, 'T': 1698491754451, 'm': True, 'M': True}
2023-10-28 16:45:55,416 - INFO - {'e': 'aggTrade', 'E': 1698491756008, 's': 'BTCUSDT', 'a': 2740955489, 'p': '34169.99000000', 'q': '0.01657000', 'f': 3260237435, 'l': 3260237435, 'T': 1698491756008, 'm': True, 'M': True}
2023-10-28 16:45:55,416 - INFO - {'e': 'aggTrade', 'E': 1698491756008, 's': 'BTCUSDT', 'a': 2740955489, 'p': '34169.99000000', 'q': '0.01657000', 'f': 3260237435, 'l': 3260237435, 'T': 1698491756008, 'm': True, 'M': True}
2023-10-28 16:45:57,484 - INFO - {'e': 'aggTrade', 'E': 1698491757998, 's': 'ETHUSDT', 'a': 949460954, 'p': '1794.72000000', 'q': '0.02780000', 'f': 1218941291, 'l': 1218941291, 'T': 1698491757998, 'm': False, 'M': True}
2023-10-28 16:45:57,486 - INFO - {'e': 'aggTrade', 'E': 1698491757998, 's': 'ETHUSDT', 'a': 949460954, 'p': '1794.72000000', 'q': '0.02780000', 'f': 1218941291, 'l': 1218941291, 'T': 1698491757998, 'm': False, 'M': True}
2023-10-28 16:45:59,548 - INFO - {'e': 'aggTrade', 'E': 1698491760576, 's': 'ETHUSDT', 'a': 949461039, 'p': '1794.71000000', 'q': '0.00900000', 'f': 1218941379, 'l': 1218941379, 'T': 1698491760576, 'm': True, 'M': True}
2023-10-28 16:45:59,553 - INFO - {'e': 'aggTrade', 'E': 1698491760576, 's': 'ETHUSDT', 'a': 949461039, 'p': '1794.71000000', 'q': '0.00900000', 'f': 1218941379, 'l': 1218941379, 'T': 1698491760576, 'm': True, 'M': True}
2023-10-28 16:46:01,610 - INFO - {'e': 'aggTrade', 'E': 1698491762716, 's': 'BTCUSDT', 'a': 2740955579, 'p': '34169.99000000', 'q': '0.01000000', 'f': 3260237536, 'l': 3260237536, 'T': 1698491762715, 'm': True, 'M': True}
2023-10-28 16:46:01,613 - INFO - {'e': 'aggTrade', 'E': 1698491762716, 's': 'BTCUSDT', 'a': 2740955579, 'p': '34169.99000000', 'q': '0.01000000', 'f': 3260237536, 'l': 3260237536, 'T': 1698491762715, 'm': True, 'M': True}
2023-10-28 16:46:03,672 - INFO - {'e': 'aggTrade', 'E': 1698491763914, 's': 'ETHUSDT', 'a': 949461043, 'p': '1794.72000000', 'q': '0.05770000', 'f': 1218941383, 'l': 1218941383, 'T': 1698491763913, 'm': False, 'M': True}
2023-10-28 16:46:03,674 - INFO - {'e': 'aggTrade', 'E': 1698491763914, 's': 'ETHUSDT', 'a': 949461043, 'p': '1794.72000000', 'q': '0.05770000', 'f': 1218941383, 'l': 1218941383, 'T': 1698491763913, 'm': False, 'M': True}
2023-10-28 16:46:05,717 - INFO - {'e': 'aggTrade', 'E': 1698491766766, 's': 'BTCUSDT', 'a': 2740955587, 'p': '34169.99000000', 'q': '0.00078000', 'f': 3260237544, 'l': 3260237544, 'T': 1698491766765, 'm': True, 'M': True}
2023-10-28 16:46:05,720 - INFO - {'e': 'aggTrade', 'E': 1698491766766, 's': 'BTCUSDT', 'a': 2740955587, 'p': '34169.99000000', 'q': '0.00078000', 'f': 3260237544, 'l': 3260237544, 'T': 1698491766765, 'm': True, 'M': True}
2023-10-28 16:46:07,759 - INFO - {'e': 'aggTrade', 'E': 1698491768852, 's': 'BTCUSDT', 'a': 2740955595, 'p': '34169.99000000', 'q': '0.02102000', 'f': 3260237553, 'l': 3260237553, 'T': 1698491768851, 'm': True, 'M': True}
2023-10-28 16:46:07,759 - INFO - {'e': 'aggTrade', 'E': 1698491768852, 's': 'BTCUSDT', 'a': 2740955595, 'p': '34169.99000000', 'q': '0.02102000', 'f': 3260237553, 'l': 3260237553, 'T': 1698491768851, 'm': True, 'M': True}
2023-10-28 16:46:09,840 - INFO - {'e': 'aggTrade', 'E': 1698491769679, 's': 'ETHUSDT', 'a': 949461050, 'p': '1794.71000000', 'q': '0.00740000', 'f': 1218941393, 'l': 1218941393, 'T': 1698491769679, 'm': True, 'M': True}
2023-10-28 16:46:09,840 - INFO - {'e': 'aggTrade', 'E': 1698491769679, 's': 'ETHUSDT', 'a': 949461050, 'p': '1794.71000000', 'q': '0.00740000', 'f': 1218941393, 'l': 1218941393, 'T': 1698491769679, 'm': True, 'M': True}
2023-10-28 16:46:11,898 - INFO - {'e': 'aggTrade', 'E': 1698491772854, 's': 'BTCUSDT', 'a': 2740955603, 'p': '34169.99000000', 'q': '0.00139000', 'f': 3260237561, 'l': 3260237561, 'T': 1698491772853, 'm': True, 'M': True}
2023-10-28 16:46:11,898 - INFO - {'e': 'aggTrade', 'E': 1698491772854, 's': 'BTCUSDT', 'a': 2740955603, 'p': '34169.99000000', 'q': '0.00139000', 'f': 3260237561, 'l': 3260237561, 'T': 1698491772853, 'm': True, 'M': True}
2023-10-28 16:46:13,941 - INFO - {'e': 'aggTrade', 'E': 1698491774744, 's': 'ETHUSDT', 'a': 949461053, 'p': '1794.71000000', 'q': '0.00420000', 'f': 1218941396, 'l': 1218941396, 'T': 1698491774744, 'm': True, 'M': True}
2023-10-28 16:46:13,941 - INFO - {'e': 'aggTrade', 'E': 1698491774744, 's': 'ETHUSDT', 'a': 949461053, 'p': '1794.71000000', 'q': '0.00420000', 'f': 1218941396, 'l': 1218941396, 'T': 1698491774744, 'm': True, 'M': True}
2023-10-28 16:46:16,021 - INFO - {'e': 'aggTrade', 'E': 1698491776870, 's': 'ETHUSDT', 'a': 949461055, 'p': '1794.72000000', 'q': '0.05570000', 'f': 1218941398, 'l': 1218941398, 'T': 1698491776870, 'm': False, 'M': True}
2023-10-28 16:46:16,024 - INFO - {'e': 'aggTrade', 'E': 1698491776870, 's': 'ETHUSDT', 'a': 949461055, 'p': '1794.72000000', 'q': '0.05570000', 'f': 1218941398, 'l': 1218941398, 'T': 1698491776870, 'm': False, 'M': True}
2023-10-28 16:46:18,082 - INFO - {'e': 'aggTrade', 'E': 1698491779083, 's': 'ETHUSDT', 'a': 949461057, 'p': '1794.71000000', 'q': '0.16230000', 'f': 1218941400, 'l': 1218941400, 'T': 1698491779082, 'm': True, 'M': True}
2023-10-28 16:46:18,082 - INFO - {'e': 'aggTrade', 'E': 1698491779083, 's': 'ETHUSDT', 'a': 949461057, 'p': '1794.71000000', 'q': '0.16230000', 'f': 1218941400, 'l': 1218941400, 'T': 1698491779082, 'm': True, 'M': True}
2023-10-28 16:46:20,137 - INFO - {'e': 'aggTrade', 'E': 1698491781248, 's': 'BTCUSDT', 'a': 2740955619, 'p': '34170.00000000', 'q': '0.00041000', 'f': 3260237577, 'l': 3260237577, 'T': 1698491781248, 'm': False, 'M': True}
2023-10-28 16:46:20,140 - INFO - {'e': 'aggTrade', 'E': 1698491781248, 's': 'BTCUSDT', 'a': 2740955619, 'p': '34170.00000000', 'q': '0.00041000', 'f': 3260237577, 'l': 3260237577, 'T': 1698491781248, 'm': False, 'M': True}
2023-10-28 16:46:22,191 - INFO - {'e': 'aggTrade', 'E': 1698491782705, 's': 'BTCUSDT', 'a': 2740955625, 'p': '34169.99000000', 'q': '0.00058000', 'f': 3260237583, 'l': 3260237583, 'T': 1698491782705, 'm': True, 'M': True}
2023-10-28 16:46:22,191 - INFO - {'e': 'aggTrade', 'E': 1698491782705, 's': 'BTCUSDT', 'a': 2740955625, 'p': '34169.99000000', 'q': '0.00058000', 'f': 3260237583, 'l': 3260237583, 'T': 1698491782705, 'm': True, 'M': True}
2023-10-28 16:46:24,243 - INFO - {'e': 'aggTrade', 'E': 1698491785237, 's': 'BTCUSDT', 'a': 2740955631, 'p': '34170.00000000', 'q': '0.09897000', 'f': 3260237589, 'l': 3260237589, 'T': 1698491785236, 'm': False, 'M': True}
2023-10-28 16:46:24,247 - INFO - {'e': 'aggTrade', 'E': 1698491785237, 's': 'BTCUSDT', 'a': 2740955631, 'p': '34170.00000000', 'q': '0.09897000', 'f': 3260237589, 'l': 3260237589, 'T': 1698491785236, 'm': False, 'M': True}
2023-10-28 16:46:26,297 - INFO - {'e': 'aggTrade', 'E': 1698491787397, 's': 'ETHUSDT', 'a': 949461071, 'p': '1794.71000000', 'q': '2.04730000', 'f': 1218941421, 'l': 1218941422, 'T': 1698491787396, 'm': True, 'M': True}
2023-10-28 16:46:26,300 - INFO - {'e': 'aggTrade', 'E': 1698491787397, 's': 'ETHUSDT', 'a': 949461071, 'p': '1794.71000000', 'q': '2.04730000', 'f': 1218941421, 'l': 1218941422, 'T': 1698491787396, 'm': True, 'M': True}
2023-10-28 16:46:28,338 - INFO - {'e': 'aggTrade', 'E': 1698491789287, 's': 'ETHUSDT', 'a': 949461076, 'p': '1794.62000000', 'q': '0.00500000', 'f': 1218941429, 'l': 1218941430, 'T': 1698491789286, 'm': False, 'M': True}
2023-10-28 16:46:28,338 - INFO - {'e': 'aggTrade', 'E': 1698491789287, 's': 'ETHUSDT', 'a': 949461076, 'p': '1794.62000000', 'q': '0.00500000', 'f': 1218941429, 'l': 1218941430, 'T': 1698491789286, 'm': False, 'M': True}
2023-10-28 16:46:30,387 - INFO - {'e': 'aggTrade', 'E': 1698491791517, 's': 'BTCUSDT', 'a': 2740955644, 'p': '34169.99000000', 'q': '0.01149000', 'f': 3260237603, 'l': 3260237603, 'T': 1698491791516, 'm': True, 'M': True}
2023-10-28 16:46:30,387 - INFO - {'e': 'aggTrade', 'E': 1698491791517, 's': 'BTCUSDT', 'a': 2740955644, 'p': '34169.99000000', 'q': '0.01149000', 'f': 3260237603, 'l': 3260237603, 'T': 1698491791516, 'm': True, 'M': True}
2023-10-28 16:46:32,439 - INFO - {'e': 'aggTrade', 'E': 1698491793576, 's': 'BTCUSDT', 'a': 2740955655, 'p': '34170.00000000', 'q': '0.15240000', 'f': 3260237616, 'l': 3260237619, 'T': 1698491793575, 'm': False, 'M': True}
2023-10-28 16:46:32,439 - INFO - {'e': 'aggTrade', 'E': 1698491793576, 's': 'BTCUSDT', 'a': 2740955655, 'p': '34170.00000000', 'q': '0.15240000', 'f': 3260237616, 'l': 3260237619, 'T': 1698491793575, 'm': False, 'M': True}
2023-10-28 16:46:34,502 - INFO - {'e': 'aggTrade', 'E': 1698491795279, 's': 'ETHUSDT', 'a': 949461083, 'p': '1794.59000000', 'q': '0.28840000', 'f': 1218941439, 'l': 1218941440, 'T': 1698491795278, 'm': True, 'M': True}
2023-10-28 16:46:34,502 - INFO - {'e': 'aggTrade', 'E': 1698491795279, 's': 'ETHUSDT', 'a': 949461083, 'p': '1794.59000000', 'q': '0.28840000', 'f': 1218941439, 'l': 1218941440, 'T': 1698491795278, 'm': True, 'M': True}
2023-10-28 16:46:36,539 - INFO - {'e': 'aggTrade', 'E': 1698491797578, 's': 'BTCUSDT', 'a': 2740955665, 'p': '34169.99000000', 'q': '0.00578000', 'f': 3260237633, 'l': 3260237633, 'T': 1698491797576, 'm': True, 'M': True}
2023-10-28 16:46:36,542 - INFO - {'e': 'aggTrade', 'E': 1698491797578, 's': 'BTCUSDT', 'a': 2740955665, 'p': '34169.99000000', 'q': '0.00578000', 'f': 3260237633, 'l': 3260237633, 'T': 1698491797576, 'm': True, 'M': True}
2023-10-28 16:46:38,596 - INFO - {'e': 'aggTrade', 'E': 1698491799709, 's': 'ETHUSDT', 'a': 949461086, 'p': '1794.59000000', 'q': '0.03860000', 'f': 1218941444, 'l': 1218941444, 'T': 1698491799709, 'm': True, 'M': True}
2023-10-28 16:46:38,598 - INFO - {'e': 'aggTrade', 'E': 1698491799709, 's': 'ETHUSDT', 'a': 949461086, 'p': '1794.59000000', 'q': '0.03860000', 'f': 1218941444, 'l': 1218941444, 'T': 1698491799709, 'm': True, 'M': True}
2023-10-28 16:46:40,650 - INFO - {'e': 'aggTrade', 'E': 1698491801786, 's': 'BTCUSDT', 'a': 2740955709, 'p': '34170.00000000', 'q': '0.00035000', 'f': 3260237694, 'l': 3260237694, 'T': 1698491801786, 'm': False, 'M': True}
2023-10-28 16:46:40,650 - INFO - {'e': 'aggTrade', 'E': 1698491801786, 's': 'BTCUSDT', 'a': 2740955709, 'p': '34170.00000000', 'q': '0.00035000', 'f': 3260237694, 'l': 3260237694, 'T': 1698491801786, 'm': False, 'M': True}
2023-10-28 16:46:42,700 - INFO - {'e': 'aggTrade', 'E': 1698491803778, 's': 'BTCUSDT', 'a': 2740955735, 'p': '34170.00000000', 'q': '0.00035000', 'f': 3260237720, 'l': 3260237720, 'T': 1698491803777, 'm': False, 'M': True}
2023-10-28 16:46:42,700 - INFO - {'e': 'aggTrade', 'E': 1698491803778, 's': 'BTCUSDT', 'a': 2740955735, 'p': '34170.00000000', 'q': '0.00035000', 'f': 3260237720, 'l': 3260237720, 'T': 1698491803777, 'm': False, 'M': True}
2023-10-28 16:46:44,758 - INFO - {'e': 'aggTrade', 'E': 1698491805896, 's': 'BTCUSDT', 'a': 2740955763, 'p': '34170.00000000', 'q': '0.00035000', 'f': 3260237749, 'l': 3260237749, 'T': 1698491805895, 'm': False, 'M': True}
2023-10-28 16:46:44,760 - INFO - {'e': 'aggTrade', 'E': 1698491805896, 's': 'BTCUSDT', 'a': 2740955763, 'p': '34170.00000000', 'q': '0.00035000', 'f': 3260237749, 'l': 3260237749, 'T': 1698491805895, 'm': False, 'M': True}
2023-10-28 16:46:46,824 - INFO - {'e': 'aggTrade', 'E': 1698491807898, 's': 'BTCUSDT', 'a': 2740955788, 'p': '34169.99000000', 'q': '0.00841000', 'f': 3260237774, 'l': 3260237774, 'T': 1698491807897, 'm': True, 'M': True}
2023-10-28 16:46:46,824 - INFO - {'e': 'aggTrade', 'E': 1698491807898, 's': 'BTCUSDT', 'a': 2740955788, 'p': '34169.99000000', 'q': '0.00841000', 'f': 3260237774, 'l': 3260237774, 'T': 1698491807897, 'm': True, 'M': True}
2023-10-28 16:46:48,889 - INFO - {'e': 'aggTrade', 'E': 1698491810029, 's': 'BTCUSDT', 'a': 2740955800, 'p': '34170.00000000', 'q': '0.00587000', 'f': 3260237786, 'l': 3260237786, 'T': 1698491810029, 'm': False, 'M': True}
2023-10-28 16:46:48,892 - INFO - {'e': 'aggTrade', 'E': 1698491810029, 's': 'BTCUSDT', 'a': 2740955800, 'p': '34170.00000000', 'q': '0.00587000', 'f': 3260237786, 'l': 3260237786, 'T': 1698491810029, 'm': False, 'M': True}
2023-10-28 16:46:50,938 - INFO - {'e': 'aggTrade', 'E': 1698491811389, 's': 'BTCUSDT', 'a': 2740955803, 'p': '34170.00000000', 'q': '0.00089000', 'f': 3260237789, 'l': 3260237789, 'T': 1698491811389, 'm': False, 'M': True}
2023-10-28 16:46:50,940 - INFO - {'e': 'aggTrade', 'E': 1698491811389, 's': 'BTCUSDT', 'a': 2740955803, 'p': '34170.00000000', 'q': '0.00089000', 'f': 3260237789, 'l': 3260237789, 'T': 1698491811389, 'm': False, 'M': True}
2023-10-28 16:46:53,006 - INFO - {'e': 'aggTrade', 'E': 1698491813840, 's': 'BTCUSDT', 'a': 2740955808, 'p': '34170.00000000', 'q': '0.01249000', 'f': 3260237794, 'l': 3260237794, 'T': 1698491813840, 'm': False, 'M': True}
2023-10-28 16:46:53,009 - INFO - {'e': 'aggTrade', 'E': 1698491813840, 's': 'BTCUSDT', 'a': 2740955808, 'p': '34170.00000000', 'q': '0.01249000', 'f': 3260237794, 'l': 3260237794, 'T': 1698491813840, 'm': False, 'M': True}
2023-10-28 16:46:55,052 - INFO - {'e': 'aggTrade', 'E': 1698491815403, 's': 'BTCUSDT', 'a': 2740955811, 'p': '34170.00000000', 'q': '0.00087000', 'f': 3260237797, 'l': 3260237797, 'T': 1698491815403, 'm': False, 'M': True}
2023-10-28 16:46:55,052 - INFO - {'e': 'aggTrade', 'E': 1698491815403, 's': 'BTCUSDT', 'a': 2740955811, 'p': '34170.00000000', 'q': '0.00087000', 'f': 3260237797, 'l': 3260237797, 'T': 1698491815403, 'm': False, 'M': True}
2023-10-28 16:46:57,085 - INFO - {'e': 'aggTrade', 'E': 1698491818160, 's': 'ETHUSDT', 'a': 949461154, 'p': '1794.16000000', 'q': '0.03850000', 'f': 1218941529, 'l': 1218941529, 'T': 1698491818160, 'm': True, 'M': True}
2023-10-28 16:46:57,085 - INFO - {'e': 'aggTrade', 'E': 1698491818160, 's': 'ETHUSDT', 'a': 949461154, 'p': '1794.16000000', 'q': '0.03850000', 'f': 1218941529, 'l': 1218941529, 'T': 1698491818160, 'm': True, 'M': True}
2023-10-28 16:46:59,124 - INFO - {'e': 'aggTrade', 'E': 1698491820030, 's': 'ETHUSDT', 'a': 949461250, 'p': '1794.16000000', 'q': '0.02560000', 'f': 1218941630, 'l': 1218941630, 'T': 1698491820030, 'm': True, 'M': True}
2023-10-28 16:46:59,126 - INFO - {'e': 'aggTrade', 'E': 1698491820030, 's': 'ETHUSDT', 'a': 949461250, 'p': '1794.16000000', 'q': '0.02560000', 'f': 1218941630, 'l': 1218941630, 'T': 1698491820030, 'm': True, 'M': True}
2023-10-28 16:47:01,190 - INFO - {'e': 'aggTrade', 'E': 1698491822255, 's': 'BTCUSDT', 'a': 2740955913, 'p': '34170.00000000', 'q': '0.00072000', 'f': 3260237909, 'l': 3260237909, 'T': 1698491822253, 'm': False, 'M': True}
2023-10-28 16:47:01,193 - INFO - {'e': 'aggTrade', 'E': 1698491822255, 's': 'BTCUSDT', 'a': 2740955913, 'p': '34170.00000000', 'q': '0.00072000', 'f': 3260237909, 'l': 3260237909, 'T': 1698491822253, 'm': False, 'M': True}
2023-10-28 16:47:03,238 - INFO - {'e': 'aggTrade', 'E': 1698491823453, 's': 'ETHUSDT', 'a': 949461254, 'p': '1794.17000000', 'q': '0.00550000', 'f': 1218941634, 'l': 1218941634, 'T': 1698491823453, 'm': False, 'M': True}
2023-10-28 16:47:03,240 - INFO - {'e': 'aggTrade', 'E': 1698491823453, 's': 'ETHUSDT', 'a': 949461254, 'p': '1794.17000000', 'q': '0.00550000', 'f': 1218941634, 'l': 1218941634, 'T': 1698491823453, 'm': False, 'M': True}
2023-10-28 16:47:05,275 - INFO - {'e': 'aggTrade', 'E': 1698491825790, 's': 'BTCUSDT', 'a': 2740955919, 'p': '34170.00000000', 'q': '0.02796000', 'f': 3260237915, 'l': 3260237915, 'T': 1698491825790, 'm': False, 'M': True}
2023-10-28 16:47:05,279 - INFO - {'e': 'aggTrade', 'E': 1698491825790, 's': 'BTCUSDT', 'a': 2740955919, 'p': '34170.00000000', 'q': '0.02796000', 'f': 3260237915, 'l': 3260237915, 'T': 1698491825790, 'm': False, 'M': True}
2023-10-28 16:47:07,338 - INFO - {'e': 'aggTrade', 'E': 1698491827745, 's': 'BTCUSDT', 'a': 2740955925, 'p': '34169.99000000', 'q': '0.05432000', 'f': 3260237921, 'l': 3260237921, 'T': 1698491827744, 'm': True, 'M': True}
2023-10-28 16:47:07,341 - INFO - {'e': 'aggTrade', 'E': 1698491827745, 's': 'BTCUSDT', 'a': 2740955925, 'p': '34169.99000000', 'q': '0.05432000', 'f': 3260237921, 'l': 3260237921, 'T': 1698491827744, 'm': True, 'M': True}
2023-10-28 16:47:09,389 - INFO - {'e': 'aggTrade', 'E': 1698491829687, 's': 'BTCUSDT', 'a': 2740955931, 'p': '34170.00000000', 'q': '0.00146000', 'f': 3260237927, 'l': 3260237927, 'T': 1698491829687, 'm': False, 'M': True}
2023-10-28 16:47:09,391 - INFO - {'e': 'aggTrade', 'E': 1698491829687, 's': 'BTCUSDT', 'a': 2740955931, 'p': '34170.00000000', 'q': '0.00146000', 'f': 3260237927, 'l': 3260237927, 'T': 1698491829687, 'm': False, 'M': True}
2023-10-28 16:47:11,417 - INFO - {'e': 'aggTrade', 'E': 1698491832491, 's': 'BTCUSDT', 'a': 2740955940, 'p': '34170.00000000', 'q': '0.00053000', 'f': 3260237938, 'l': 3260237938, 'T': 1698491832490, 'm': False, 'M': True}
2023-10-28 16:47:11,417 - INFO - {'e': 'aggTrade', 'E': 1698491832491, 's': 'BTCUSDT', 'a': 2740955940, 'p': '34170.00000000', 'q': '0.00053000', 'f': 3260237938, 'l': 3260237938, 'T': 1698491832490, 'm': False, 'M': True}
2023-10-28 16:47:13,479 - INFO - {'e': 'aggTrade', 'E': 1698491833734, 's': 'BTCUSDT', 'a': 2740955946, 'p': '34170.00000000', 'q': '0.00030000', 'f': 3260237944, 'l': 3260237944, 'T': 1698491833733, 'm': False, 'M': True}
2023-10-28 16:47:13,483 - INFO - {'e': 'aggTrade', 'E': 1698491833734, 's': 'BTCUSDT', 'a': 2740955946, 'p': '34170.00000000', 'q': '0.00030000', 'f': 3260237944, 'l': 3260237944, 'T': 1698491833733, 'm': False, 'M': True}
2023-10-28 16:47:15,543 - INFO - {'e': 'aggTrade', 'E': 1698491836480, 's': 'BTCUSDT', 'a': 2740955949, 'p': '34170.00000000', 'q': '0.01200000', 'f': 3260237947, 'l': 3260237947, 'T': 1698491836480, 'm': False, 'M': True}
2023-10-28 16:47:15,546 - INFO - {'e': 'aggTrade', 'E': 1698491836480, 's': 'BTCUSDT', 'a': 2740955949, 'p': '34170.00000000', 'q': '0.01200000', 'f': 3260237947, 'l': 3260237947, 'T': 1698491836480, 'm': False, 'M': True}
2023-10-28 16:47:17,611 - INFO - {'e': 'aggTrade', 'E': 1698491838367, 's': 'BTCUSDT', 'a': 2740955961, 'p': '34169.99000000', 'q': '0.00545000', 'f': 3260237973, 'l': 3260237973, 'T': 1698491838367, 'm': True, 'M': True}
2023-10-28 16:47:17,614 - INFO - {'e': 'aggTrade', 'E': 1698491838367, 's': 'BTCUSDT', 'a': 2740955961, 'p': '34169.99000000', 'q': '0.00545000', 'f': 3260237973, 'l': 3260237973, 'T': 1698491838367, 'm': True, 'M': True}
2023-10-28 16:47:19,660 - INFO - {'e': 'aggTrade', 'E': 1698491840322, 's': 'BTCUSDT', 'a': 2740955963, 'p': '34169.99000000', 'q': '0.00160000', 'f': 3260237975, 'l': 3260237975, 'T': 1698491840322, 'm': True, 'M': True}
2023-10-28 16:47:19,663 - INFO - {'e': 'aggTrade', 'E': 1698491840322, 's': 'BTCUSDT', 'a': 2740955963, 'p': '34169.99000000', 'q': '0.00160000', 'f': 3260237975, 'l': 3260237975, 'T': 1698491840322, 'm': True, 'M': True}
2023-10-28 16:47:21,704 - INFO - {'e': 'aggTrade', 'E': 1698491842652, 's': 'BTCUSDT', 'a': 2740955966, 'p': '34169.99000000', 'q': '0.00293000', 'f': 3260237978, 'l': 3260237978, 'T': 1698491842651, 'm': True, 'M': True}
2023-10-28 16:47:21,707 - INFO - {'e': 'aggTrade', 'E': 1698491842652, 's': 'BTCUSDT', 'a': 2740955966, 'p': '34169.99000000', 'q': '0.00293000', 'f': 3260237978, 'l': 3260237978, 'T': 1698491842651, 'm': True, 'M': True}
2023-10-28 19:58:10,237 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:10,237 - ERROR - No data available
2023-10-28 19:58:12,290 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:12,291 - ERROR - No data available
2023-10-28 19:58:14,358 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:14,358 - ERROR - No data available
2023-10-28 19:58:16,392 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:16,393 - ERROR - No data available
2023-10-28 19:58:18,438 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:18,440 - ERROR - No data available
2023-10-28 19:58:20,490 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:20,490 - ERROR - No data available
2023-10-28 19:58:22,536 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:22,536 - ERROR - No data available
2023-10-28 19:58:24,581 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:24,581 - ERROR - No data available
2023-10-28 19:58:26,624 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:26,624 - ERROR - No data available
2023-10-28 19:58:28,668 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:28,668 - ERROR - No data available
2023-10-28 19:58:30,720 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:30,721 - ERROR - No data available
2023-10-28 19:58:32,755 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:32,755 - ERROR - No data available
2023-10-28 19:58:34,810 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:34,811 - ERROR - No data available
2023-10-28 19:58:36,875 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:36,876 - ERROR - No data available
2023-10-28 19:58:38,908 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:38,909 - ERROR - No data available
2023-10-28 19:58:40,959 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:40,959 - ERROR - No data available
2023-10-28 19:58:43,014 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:43,014 - ERROR - No data available
2023-10-28 19:58:45,063 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:45,064 - ERROR - No data available
2023-10-28 19:58:47,113 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:47,114 - ERROR - No data available
2023-10-28 19:58:49,176 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:49,177 - ERROR - No data available
2023-10-28 19:58:51,242 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:51,243 - ERROR - No data available
2023-10-28 19:58:53,280 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:53,280 - ERROR - No data available
2023-10-28 19:58:55,331 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:55,331 - ERROR - No data available
2023-10-28 19:58:57,375 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:57,375 - ERROR - No data available
2023-10-28 19:58:59,432 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:58:59,433 - ERROR - No data available
2023-10-28 19:59:01,488 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:01,489 - ERROR - No data available
2023-10-28 19:59:03,549 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:03,549 - ERROR - No data available
2023-10-28 19:59:05,584 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:05,584 - ERROR - No data available
2023-10-28 19:59:07,640 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:07,641 - ERROR - No data available
2023-10-28 19:59:09,711 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:09,711 - ERROR - No data available
2023-10-28 19:59:11,755 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:11,755 - ERROR - No data available
2023-10-28 19:59:13,798 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:13,799 - ERROR - No data available
2023-10-28 19:59:15,847 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:15,848 - ERROR - No data available
2023-10-28 19:59:17,892 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:17,893 - ERROR - No data available
2023-10-28 19:59:19,940 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:19,940 - ERROR - No data available
2023-10-28 19:59:21,972 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:21,972 - ERROR - No data available
2023-10-28 19:59:24,037 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:24,038 - ERROR - No data available
2023-10-28 19:59:26,071 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:26,072 - ERROR - No data available
2023-10-28 19:59:28,120 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:28,120 - ERROR - No data available
2023-10-28 19:59:30,180 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:30,181 - ERROR - No data available
2023-10-28 19:59:32,236 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:32,236 - ERROR - No data available
2023-10-28 19:59:34,300 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:34,300 - ERROR - No data available
2023-10-28 19:59:36,328 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:36,329 - ERROR - No data available
2023-10-28 19:59:38,379 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:38,380 - ERROR - No data available
2023-10-28 19:59:40,426 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:40,426 - ERROR - No data available
2023-10-28 19:59:42,476 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:42,477 - ERROR - No data available
2023-10-28 19:59:44,518 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:44,518 - ERROR - No data available
2023-10-28 19:59:46,566 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:46,566 - ERROR - No data available
2023-10-28 19:59:48,608 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:48,608 - ERROR - No data available
2023-10-28 19:59:50,661 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:50,661 - ERROR - No data available
2023-10-28 19:59:52,708 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:52,709 - ERROR - No data available
2023-10-28 19:59:54,752 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:54,753 - ERROR - No data available
2023-10-28 19:59:56,809 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:56,810 - ERROR - No data available
2023-10-28 19:59:58,873 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 19:59:58,873 - ERROR - No data available
2023-10-28 20:00:00,915 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:00,915 - ERROR - No data available
2023-10-28 20:00:02,966 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:02,967 - ERROR - No data available
2023-10-28 20:00:05,012 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:05,012 - ERROR - No data available
2023-10-28 20:00:07,063 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:07,064 - ERROR - No data available
2023-10-28 20:00:09,119 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:09,119 - ERROR - No data available
2023-10-28 20:00:11,157 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:11,157 - ERROR - No data available
2023-10-28 20:00:13,207 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:13,208 - ERROR - No data available
2023-10-28 20:00:15,260 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:15,260 - ERROR - No data available
2023-10-28 20:00:17,296 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:17,297 - ERROR - No data available
2023-10-28 20:00:19,341 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:19,341 - ERROR - No data available
2023-10-28 20:00:21,399 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:21,399 - ERROR - No data available
2023-10-28 20:00:23,452 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:23,453 - ERROR - No data available
2023-10-28 20:00:25,492 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:25,492 - ERROR - No data available
2023-10-28 20:00:27,527 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:27,528 - ERROR - No data available
2023-10-28 20:00:29,563 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:29,563 - ERROR - No data available
2023-10-28 20:00:31,602 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:31,603 - ERROR - No data available
2023-10-28 20:00:33,648 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:33,649 - ERROR - No data available
2023-10-28 20:00:35,690 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:35,690 - ERROR - No data available
2023-10-28 20:00:37,742 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:37,742 - ERROR - No data available
2023-10-28 20:00:39,782 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:39,783 - ERROR - No data available
2023-10-28 20:00:41,833 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:41,833 - ERROR - No data available
2023-10-28 20:00:43,864 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:43,864 - ERROR - No data available
2023-10-28 20:00:45,918 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:45,919 - ERROR - No data available
2023-10-28 20:00:47,968 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:47,968 - ERROR - No data available
2023-10-28 20:00:50,024 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:50,025 - ERROR - No data available
2023-10-28 20:00:52,094 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:52,094 - ERROR - No data available
2023-10-28 20:00:54,158 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:54,159 - ERROR - No data available
2023-10-28 20:00:56,226 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:56,227 - ERROR - No data available
2023-10-28 20:00:58,270 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:00:58,271 - ERROR - No data available
2023-10-28 20:01:00,314 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:00,314 - ERROR - No data available
2023-10-28 20:01:02,349 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:02,349 - ERROR - No data available
2023-10-28 20:01:04,386 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:04,386 - ERROR - No data available
2023-10-28 20:01:06,437 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:06,438 - ERROR - No data available
2023-10-28 20:01:08,489 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:08,489 - ERROR - No data available
2023-10-28 20:01:10,517 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:10,517 - ERROR - No data available
2023-10-28 20:01:12,572 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:12,572 - ERROR - No data available
2023-10-28 20:01:14,635 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:14,636 - ERROR - No data available
2023-10-28 20:01:16,703 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:16,704 - ERROR - No data available
2023-10-28 20:01:18,756 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:18,757 - ERROR - No data available
2023-10-28 20:01:20,808 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:20,809 - ERROR - No data available
2023-10-28 20:01:22,827 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:22,827 - ERROR - No data available
2023-10-28 20:01:24,862 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:24,862 - ERROR - No data available
2023-10-28 20:01:26,914 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:26,915 - ERROR - No data available
2023-10-28 20:01:28,962 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:28,963 - ERROR - No data available
2023-10-28 20:01:31,014 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:31,014 - ERROR - No data available
2023-10-28 20:01:33,067 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:33,067 - ERROR - No data available
2023-10-28 20:01:35,110 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:35,111 - ERROR - No data available
2023-10-28 20:01:37,143 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:37,144 - ERROR - No data available
2023-10-28 20:01:39,189 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:39,190 - ERROR - No data available
2023-10-28 20:01:41,243 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:41,243 - ERROR - No data available
2023-10-28 20:01:43,281 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:43,282 - ERROR - No data available
2023-10-28 20:01:45,333 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:45,333 - ERROR - No data available
2023-10-28 20:01:47,386 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:47,386 - ERROR - No data available
2023-10-28 20:01:49,441 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:49,441 - ERROR - No data available
2023-10-28 20:01:51,506 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:51,507 - ERROR - No data available
2023-10-28 20:01:53,569 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:53,570 - ERROR - No data available
2023-10-28 20:01:55,631 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:55,632 - ERROR - No data available
2023-10-28 20:01:57,653 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:57,654 - ERROR - No data available
2023-10-28 20:01:59,713 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:01:59,714 - ERROR - No data available
2023-10-28 20:02:01,780 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:01,781 - ERROR - No data available
2023-10-28 20:02:03,834 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:03,835 - ERROR - No data available
2023-10-28 20:02:05,870 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:05,870 - ERROR - No data available
2023-10-28 20:02:07,923 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:07,923 - ERROR - No data available
2023-10-28 20:02:09,959 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:09,960 - ERROR - No data available
2023-10-28 20:02:12,011 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:12,011 - ERROR - No data available
2023-10-28 20:02:14,050 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:14,050 - ERROR - No data available
2023-10-28 20:02:16,106 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:16,106 - ERROR - No data available
2023-10-28 20:02:18,164 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:18,164 - ERROR - No data available
2023-10-28 20:02:20,218 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:20,218 - ERROR - No data available
2023-10-28 20:02:22,276 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:22,277 - ERROR - No data available
2023-10-28 20:02:24,328 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:24,328 - ERROR - No data available
2023-10-28 20:02:26,364 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:26,365 - ERROR - No data available
2023-10-28 20:02:28,421 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:28,421 - ERROR - No data available
2023-10-28 20:02:30,454 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:30,454 - ERROR - No data available
2023-10-28 20:02:32,497 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:32,498 - ERROR - No data available
2023-10-28 20:02:34,550 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:34,551 - ERROR - No data available
2023-10-28 20:02:36,610 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:36,610 - ERROR - No data available
2023-10-28 20:02:38,658 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:38,658 - ERROR - No data available
2023-10-28 20:02:40,704 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:40,705 - ERROR - No data available
2023-10-28 20:02:42,768 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:42,769 - ERROR - No data available
2023-10-28 20:02:44,788 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:44,788 - ERROR - No data available
2023-10-28 20:02:46,829 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:46,830 - ERROR - No data available
2023-10-28 20:02:48,866 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:48,866 - ERROR - No data available
2023-10-28 20:02:50,905 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:50,906 - ERROR - No data available
2023-10-28 20:02:52,956 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:52,957 - ERROR - No data available
2023-10-28 20:02:55,017 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:55,018 - ERROR - No data available
2023-10-28 20:02:57,078 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:57,079 - ERROR - No data available
2023-10-28 20:02:59,145 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:02:59,146 - ERROR - No data available
2023-10-28 20:03:01,192 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:03:01,193 - ERROR - No data available
2023-10-28 20:03:03,241 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:03:03,242 - ERROR - No data available
2023-10-28 20:03:05,277 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:03:05,278 - ERROR - No data available
2023-10-28 20:03:07,325 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:03:07,326 - ERROR - No data available
2023-10-28 20:03:09,389 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:03:09,390 - ERROR - No data available
2023-10-28 20:03:11,420 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:03:11,420 - ERROR - No data available
2023-10-28 20:03:13,462 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:03:13,462 - ERROR - No data available
2023-10-28 20:03:15,515 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:03:15,515 - ERROR - No data available
2023-10-28 20:03:17,562 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:03:17,562 - ERROR - No data available
2023-10-28 20:03:19,607 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 20:03:19,607 - ERROR - No data available
2023-10-28 21:06:40,283 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 21:06:40,284 - ERROR - No data available
2023-10-28 21:06:42,332 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 21:06:42,333 - ERROR - No data available
2023-10-28 21:06:44,391 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 21:06:44,391 - ERROR - No data available
2023-10-28 21:06:46,443 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 21:06:46,443 - ERROR - No data available
2023-10-28 21:06:48,487 - ERROR - Exception Occured Expecting value: line 1 column 1 (char 0)
2023-10-28 21:06:48,488 - ERROR - No data available
